<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML+RDFa 1.0//EN" "http://www.w3.org/MarkUp/DTD/xhtml-rdfa-1.dtd">

<!-- paulirish.com/2008/conditional-stylesheets-vs-css-hacks-answer-neither/ -->
<!--[if lt IE 7]> <html class="no-js ie6 ie" xmlns="http://www.w3.org/1999/xhtml" lang="en" dir="ltr"> <![endif]-->
<!--[if IE 7]>    <html class="no-js ie7 ie" xmlns="http://www.w3.org/1999/xhtml" lang="en" dir="ltr"> <![endif]-->
<!--[if IE 8]>    <html class="no-js ie8 ie" xmlns="http://www.w3.org/1999/xhtml" lang="en" dir="ltr"> <![endif]-->
<!--[if IE 9]>    <html class="no-js ie9 ie" xmlns="http://www.w3.org/1999/xhtml" lang="en" dir="ltr"> <![endif]-->
<!--[if gt IE 9]><!--> <html class="no-js" xmlns="http://www.w3.org/1999/xhtml" lang="en" dir="ltr" 
  xmlns:content="http://purl.org/rss/1.0/modules/content/"
  xmlns:dc="http://purl.org/dc/terms/"
  xmlns:foaf="http://xmlns.com/foaf/0.1/"
  xmlns:og="http://ogp.me/ns#"
  xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
  xmlns:sioc="http://rdfs.org/sioc/ns#"
  xmlns:sioct="http://rdfs.org/sioc/types#"
  xmlns:skos="http://www.w3.org/2004/02/skos/core#"
  xmlns:xsd="http://www.w3.org/2001/XMLSchema#"> <!--<![endif]-->


<!-- Mirrored from legacy.vectorbase.org/glossary by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 10 May 2021 21:54:50 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="shortcut icon" href="images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
  <title>Glossary | VectorBase</title>
  <link type="text/css" rel="stylesheet" href="sites/default/files/ftp/css/css_xE-rWrJf-fncB6ztZfd2huxqgxu4WO-qwma6Xer30m4.css" media="all" />
<link type="text/css" rel="stylesheet" href="sites/default/files/ftp/css/css_mxbvRAV0_YoXwGCXbsQzCkegc1pXHICzfd8PPQD9YE4.css" media="all" />
<link type="text/css" rel="stylesheet" href="sites/default/files/ftp/css/css_PICr0GKGg7bWv71lKRa_OEkomgs62T8kc1nkoHGPaHQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="sites/default/files/ftp/css/css_eufK8PJpxJVaRetPZyI9u26Sa07yMCpM7nLkBEvpRHA.css" media="all" />
<link type="text/css" rel="stylesheet" href="sites/default/files/ftp/css/css_cP6Vh1gUqUILwv-7MkMGbDM2EBjPjnFQHo4vhLrssf8.css" media="all" />
<link type="text/css" rel="stylesheet" href="sites/default/files/ftp/css/css_rprouKRjRbkIKo9d6RXSljtPMncKZiBQLHESwU_LbNM.css" media="all" />
    <script type="text/javascript" src="sites/default/files/ftp/js/js_gabcqQuKQ3_nd37NqOuWd9WQRIYOjCKdKTvRCaIo06I.js"></script>
<script type="text/javascript" src="sites/default/files/ftp/js/js_zJeG4OGffO4aampI_xLmtj3WRIcE06V9UF8xhvTJKAM.js"></script>
<script type="text/javascript" src="sites/default/files/ftp/js/js_cLEvhoD3_g4lSZ6lfYzcDtmVdms3cZizAjb5jA3ABGQ.js"></script>
<script type="text/javascript" src="sites/default/files/ftp/js/js_08ByJhRYjSGp07AndqqBMgkJe1tY-8dUHOOO8Jy0NMY.js"></script>
<script type="text/javascript" src="sites/default/files/ftp/js/js_mw06s-gYd-ANxp8zZ_N-xVkvD8F3Tf5OMGUGwU-H6zY.js"></script>
<script type="text/javascript" src="sites/default/files/ftp/js/js_EXqXMFbvpy4r2fvHe4U7HNe5y1ZFdcqUrqUlbWUBfb4.js"></script>
<script type="text/javascript" src="sites/default/files/ftp/js/js_rsGiM5M1ffe6EhN-RnhM5f3pDyJ8ZAPFJNKpfjtepLk.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","https://www.google-analytics.com/analytics.js","ga");ga("create", "UA-6417661-1", {"cookieDomain":".legacy.vectorbase.org"});// URL decode, just in case 
var hrefResult=decodeURIComponent(window.location.href);
var pathResult=decodeURIComponent(window.location.pathname);
// check URL has search
if (pathResult.includes("https://legacy.vectorbase.org/search/site/")) {
	// split pathname with /, then take the last value as search/query parameter
	var splitPathresult=pathResult.split("index.html");
	var searchValue=splitPathresult[splitPathresult.length-1];
	//strip quotes from searchValue
	searchValue=searchValue.replace(/['"]+/g, '');
	// in case Category (?) exists - needs to check .href, not .pathname
	if (hrefResult.includes("?")) {
		// split pathname with /, then take the last value as search/query parameter
		var splitPathresult=pathResult.split("index.html");
		var searchValue=splitPathresult[splitPathresult.length-1];
		// split pathname with ?, then take the last value as GA (Google Analytics) category
		var splitHref=hrefResult.split("?");
		var categoryValue=splitHref[splitHref.length-1];				
		// if the first character of Category is ampersand, &, remove it, just in case
		if (categoryValue[0] == "&") {
			categoryValue = categoryValue.substr(1);
		}
		//strip quotes from categoryValue		
		categoryValue=categoryValue.replace(/['"]+/g, '');
		// make ga command
		var gaCommandUrl = '/search/site?search=' + searchValue + '&' + categoryValue;
	    // set page instead of send pageview
	    ga('set', 'page', gaCommandUrl);
	} else {
		var gaCommandUrl = '/search/site?search=' + searchValue;
		// set page instead of send pageview
	    ga('set', 'page', gaCommandUrl);
	}
}ga("send", "pageview");
//--><!]]>
</script>
<script type="text/javascript" src="sites/default/files/ftp/js/js_Cv5tN7wsTPTtQjF3wZdxqcOLDGuxEFfWQy50Tyq9mEw.js"></script>
<script type="text/javascript" src="sites/default/files/ftp/js/js_tl9xHQ_hdQMKNylSuI2Op9wFOQu73kVboIrZvWiiTwY.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"acquia_marina","theme_token":"a4Ks1BCEB1KGBd_HGKFXulbVHp2T4GJNvL6jCGkwtU4","js":{"public:\/\/google_tag\/google_tag.script.js":1,"sites\/all\/modules\/added\/devel\/jquery-1.4.4-uncompressed.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"misc\/ui\/jquery.ui.core.min.js":1,"misc\/ui\/jquery.ui.widget.min.js":1,"misc\/ui\/jquery.ui.position.min.js":1,"misc\/ui\/jquery.ui.autocomplete.min.js":1,"misc\/jquery.cookie.js":1,"misc\/jquery.form.js":1,"misc\/ajax.js":1,"sites\/all\/libraries\/jquery\/jquery-1.10.2.min.js":1,"sites\/all\/modules\/added\/jqmulti\/js\/switch.js":1,"sites\/all\/modules\/added\/attach\/attach.js":1,"sites\/all\/modules\/added\/google_analytics\/googleanalytics.js":1,"0":1,"misc\/progress.js":1,"modules\/file\/file.js":1,"sites\/all\/modules\/custom\/vbsearch\/js\/vbsearch_basic.js":1,"sites\/all\/modules\/custom\/vbsearch\/modules\/vbsearch_autocomplete\/vbsearch_autocomplete.js":1,"sites\/all\/libraries\/superfish\/jquery.hoverIntent.minified.js":1,"sites\/all\/libraries\/superfish\/superfish.js":1,"sites\/all\/libraries\/superfish\/supersubs.js":1,"sites\/all\/modules\/added\/superfish\/superfish.js":1,"sites\/all\/themes\/fusion\/fusion_core\/js\/script.js":1,"sites\/all\/themes\/acquia_marina\/js\/jquery.browser.js":1,"sites\/all\/themes\/acquia_marina\/js\/advanced-search.js":1,"sites\/all\/themes\/acquia_marina\/js\/jquery.corner.js":1,"sites\/all\/themes\/acquia_marina\/js\/acquia-marina-script.js":1,"sites\/all\/themes\/acquia_marina\/js\/tablesorter\/jquery.tablesorter.js":1,"sites\/all\/themes\/acquia_marina\/js\/jquery.animate-shadow-min.js":1},"css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.autocomplete.css":1,"sites\/all\/modules\/added\/attach\/attach.css":1,"sites\/all\/modules\/added\/calendar\/css\/calendar_multiday.css":1,"modules\/comment\/comment.css":1,"sites\/all\/modules\/custom\/cvbrowser\/css\/cvbrowser.css":1,"sites\/all\/modules\/added\/date\/date_api\/date.css":1,"sites\/all\/modules\/added\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/added\/views\/css\/views.css":1,"sites\/all\/modules\/added\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/custom\/error_log\/error_log.css":1,"sites\/all\/modules\/custom\/fast_gallery\/fast_gallery.css":1,"sites\/all\/modules\/custom\/iptc_editor\/iptc_editor.css":1,"sites\/all\/modules\/custom\/lexicon\/css\/lexicon.css":1,"sites\/all\/modules\/custom\/submenu_arrows\/submenu_arrows.css":1,"sites\/all\/modules\/custom\/rna_seq\/css\/rna_seq.css":1,"modules\/file\/file.css":1,"file":1,"sites\/all\/modules\/custom\/vbsearch\/css\/vbsearch.css":1,"sites\/all\/modules\/custom\/vbsearch\/modules\/vbsearch_autocomplete\/vbsearch_autocomplete.css":1,"sites\/all\/libraries\/superfish\/css\/superfish.css":1,"sites\/all\/libraries\/superfish\/style\/default.css":1,"sites\/all\/themes\/fusion\/fusion_core\/css\/fusion-style.css":1,"sites\/all\/themes\/fusion\/fusion_core\/css\/fusion-typography.css":1,"sites\/all\/themes\/fusion\/fusion_core\/skins\/core\/fusion-core-skins.css":1,"sites\/all\/themes\/acquia_marina\/css\/search.css":1,"sites\/all\/themes\/acquia_marina\/css\/fusion-acquia-marina-style.css":1,"sites\/all\/themes\/acquia_marina\/css\/slideshow.css":1,"sites\/all\/themes\/fusion\/fusion_core\/css\/grid16-960.css":1,"sites\/all\/themes\/acquia_marina\/css\/local.css":1}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackDomainMode":1},"blast":{"blastPath":"sites\/all\/modules\/custom\/blast"},"clustalw":{"clustalwPath":"sites\/all\/modules\/custom\/clustalw"},"hmmer":{"hmmerPath":"sites\/all\/modules\/custom\/hmmer"},"xgrid":{"xgridPath":"sites\/all\/modules\/custom\/xgrid"},"file":{"elements":{"#edit-field-file-search-und-0-upload":"txt"}},"ajax":{"edit-field-file-search-und-0-upload-button":{"wrapper":"edit-field-file-search-und-0-ajax-wrapper","effect":"fade","progress":{"type":"throbber","message":null},"event":"mousedown","keypress":true,"prevent":"click","url":"https:\/\/legacy.vectorbase.org\/file\/ajax\/file_search\/field_file_search\/und\/0\/form-sFD5rGiHK8cn7BWRorlD5vOF25okb3W1fCFBHq0MonY","submit":{"_triggering_element_name":"field_file_search_und_0_upload_button","_triggering_element_value":"Upload"}}},"urlIsAjaxTrusted":{"https:\/\/legacy.vectorbase.org\/file\/ajax\/file_search\/field_file_search\/und\/0\/form-sFD5rGiHK8cn7BWRorlD5vOF25okb3W1fCFBHq0MonY":true,"\/glossary":true,"https:\/\/legacy.vectorbase.org\/user?destination=glossary":true},"superfish":{"1":{"id":"1","sf":{"animation":{"opacity":"show"},"speed":"\u0027fast\u0027","autoArrows":false,"dropShadows":false,"disableHI":false},"plugins":{"supposition":false,"bgiframe":false,"supersubs":{"minWidth":"12","maxWidth":"27","extraWidth":1}}}}});
//--><!]]>
</script>
</head>
<body id="pid-glossary" class="html not-front not-logged-in no-sidebars page-glossary win gecko font-size-12 grid-type-960 grid-width-16 sidebars-split" >
  <div id="skip-link">
    <a href="#main-content-area">Skip to main content area</a>
  </div>
  
<!-- page-top region -->
  <div id="page-top" class="region region-page-top page-top">
    <div id="page-top-inner" class="page-top-inner inner">
      <noscript aria-hidden="true"><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-N8WDMVJ" height="0" width="0" style="display:none;visibility:hidden" title="Google Tag Manager">Google Tag Manager</iframe></noscript>    </div><!-- /page-top-inner -->
  </div><!-- /page-top -->
    <div id="page" class="page">
    <div id="page-inner" class="page-inner">
      
      <!-- header-group region: width = grid_width -->
      <div id="header-group-wrapper" class="header-group-wrapper full-width clearfix">
        <div id="header-group" class="header-group region grid16-16">
          <div id="header-group-inner" class="header-group-inner inner clearfix">

                        <div id="header-site-info" class="header-site-info">
              <div id="header-site-info-inner" class="header-site-info-inner gutter">
                                <div id="logo">
                  <a href="index.html" title="Home"><img src="sites/default/files/ftp/images/dot_ked_100_trans.png" alt="Home" /></a>
                </div>
                                                <div id="site-name-wrapper" class="clearfix">
                                                          <div id="site-name"><a href="index.html" title="Home" rel="home">VectorBase</a></div>
                                                                          <span id="slogan">Bioinformatics Resource for Invertebrate Vectors of Human Pathogens</span>
                                  </div><!-- /site-name-wrapper -->
                              </div><!-- /header-site-info-inner -->
            </div><!-- /header-site-info -->
            
            <div id="block-search-form" class="block block-search my_search first  odd">
  <div class="inner clearfix gutter">
  	<div class="inner-wrapper">
      <div class="inner-inner">
                        <div class="content clearfix">
        <form enctype="multipart/form-data" action="https://legacy.vectorbase.org/glossary" method="post" id="search-block-form" accept-charset="UTF-8"><div>
<div id="search" class="container-inline">
  <div id="search-inner">
            <div class="form-item form-type-textfield form-item-search-block-form">
  <label class="element-invisible" for="edit-search-block-form--2">Search </label>
 <input title="Enter the terms you wish to search for." placeholder="Enter search terms" type="text" id="edit-search-block-form--2" name="search_block_form" value="" size="15" maxlength="128" class="form-text" />
</div>
<div class="field-type-file field-name-field-file-search field-widget-file-generic form-wrapper" id="edit-field-file-search"><div id="edit-field-file-search-und-0-ajax-wrapper"><div class="form-item form-type-managed-file form-item-field-file-search-und-0">
  <label for="edit-field-file-search-und-0-upload">File Search </label>
 <div class="file-widget form-managed-file clearfix"><input type="file" id="edit-field-file-search-und-0-upload" name="files[field_file_search_und_0]" size="22" class="form-file" /><input type="submit" id="edit-field-file-search-und-0-upload-button" name="field_file_search_und_0_upload_button" value="Upload" class="form-submit" /><input type="hidden" name="field_file_search[und][0][fid]" value="0" />
<input type="hidden" name="field_file_search[und][0][display]" value="1" />
</div>
<div class="description">Files must be less than <strong>5 KB</strong>.<br />Allowed file types: <strong>txt</strong>.</div>
</div>
</div></div><div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Go" class="form-submit" /><a href="https://legacy.vectorbase.org/search/site/%2A?as=True" id="advanced-search-link">Advanced Search</a><a href="# " id="edit-toggle-basic-search-type">Switch Search Type</a></div><input type="hidden" name="form_build_id" value="form-sFD5rGiHK8cn7BWRorlD5vOF25okb3W1fCFBHq0MonY" />
<input type="hidden" name="form_id" value="search_block_form" />
  </div>
</div>
</div></form>      </div>
      </div><!-- /inner-inner -->
	  </div><!-- /inner-wrapper -->
  </div><!-- /block-inner -->
</div><!-- /block --><div id="block-login_block-login_block" class="block block-login-block  last even">
  <div class="inner clearfix gutter">
  	<div class="inner-wrapper">
      <div class="inner-inner">
                        <div class="content clearfix">
        <form action="https://legacy.vectorbase.org/user?destination=glossary" method="post" id="login-block-form" accept-charset="UTF-8"><div><input type="hidden" name="form_build_id" value="form-7z1mhkr9FYOuOsnY7Z2AE69G50EkNULBTlSwyfS6FE8" />
<input type="hidden" name="form_id" value="login_block_block" />
<div class="form-actions form-wrapper" id="edit-actions--2"><input type="submit" id="edit-submit--2" name="op" value="Login" class="form-submit" /></div></div></form>      </div>
      </div><!-- /inner-inner -->
	  </div><!-- /inner-wrapper -->
  </div><!-- /block-inner -->
</div><!-- /block --><div id="content-messages-inner" class="content-messages-inner gutter"><div class="messages error" style="clear:both;">This legacy VectorBase site will be shut down at the end of May 2021. Please visit our <a rel="noreferrer" href="https://vectorbase.org/">new website</a> as part of VEuPathDB.</div></div>          </div><!-- /header-group-inner -->

          
<!-- main-menu region -->
<div id="main-menu-wrapper" class="main-menu-wrapper full-width clearfix">
  <div id="main-menu" class="region region-main-menu main-menu  grid16-16">
    <div id="main-menu-inner" class="main-menu-inner inner">
      <div id="block-superfish-1" class="block block-superfish first last odd">
  <div class="inner clearfix gutter">
  	<div class="inner-wrapper">
      <div class="inner-inner">
                        <div class="content clearfix">
        <ul id="superfish-1" class="menu sf-menu sf-navigation sf-horizontal sf-style-default"><li id="menu-9388-1" class="first odd sf-item-1 sf-depth-1 sf-no-children"><a href="zika.html" title="Zika Resource Page link" class="sf-depth-1">Zika</a></li><li id="menu-1005-1" class="middle even sf-item-2 sf-depth-1 menuparent"><a href="about.html" title="Information about this invertebrate vector Bioinformatics Resource Center (BRC) " class="sf-depth-1 menuparent">About</a><ul><li id="menu-3846-1" class="first odd sf-item-1 sf-depth-2 sf-no-children"><a href="swg.html" title="Details of VectorBase Scientific Working Group" class="sf-depth-2">VectorBase Scientific Working Group</a></li><li id="menu-3847-1" class="middle even sf-item-2 sf-depth-2 sf-no-children"><a href="sug.html" title="Details of VectorBase Scientific User Group" class="sf-depth-2">VectorBase Scientific User Group</a></li><li id="menu-3848-1" class="middle odd sf-item-3 sf-depth-2 sf-no-children"><a href="personnel.html" title="Details of Personnel" class="sf-depth-2">Personnel</a></li><li id="menu-14054-1" class="last even sf-item-4 sf-depth-2 sf-no-children"><a href="privacy.html" class="sf-depth-2">Data Privacy</a></li></ul></li><li id="menu-962-1" class="middle odd sf-item-3 sf-depth-1 menuparent"><a href="organisms.html" title="" class="sf-depth-1 menuparent">Organisms</a><ul><li id="menu-6928-1" class="first odd sf-item-1 sf-depth-2 menuparent"><a href="taxonomy/aedes.html" title="" class="sf-depth-2 menuparent"><table class="menuarrow"><tr><td class="menuarrowtext"><i>Aedes</i> </td><td class="menuarrowicon"><img class="menuarrow" src="sites/default/files/ftp/triang.png" alt="submenu arrow"></td></tr></table></a><ul class="sf-megamenu"><li class="sf-megamenu-wrapper first odd sf-item-1 sf-depth-2 menuparent"><ol><li id="menu-781-1" class="first odd sf-item-1 sf-depth-3 sf-no-children"><a href="organisms/aedes-aegypti.html" title="" class="sf-depth-3"><i>Aedes aegypti</i></a></li><li id="menu-6929-1" class="last even sf-item-2 sf-depth-3 sf-no-children"><a href="organisms/aedes-albopictus.html" class="sf-depth-3"><i>Aedes albopictus</i></a></li></ol></li></ul></li><li id="menu-3164-1" class="middle even sf-item-2 sf-depth-2 menuparent"><a href="taxonomy/anopheles.html" title="" class="sf-depth-2 menuparent"><table class="menuarrow"><tr><td class="menuarrowtext"><i>Anopheles</i> </td><td class="menuarrowicon"><img class="menuarrow" src="sites/default/files/ftp/triang.png" alt="submenu arrow"></td></tr></table></a><ul class="sf-megamenu"><li class="sf-megamenu-wrapper middle even sf-item-2 sf-depth-2 menuparent"><ol><li id="menu-3156-1" class="first odd sf-item-1 sf-depth-3 sf-no-children"><a href="organisms/anopheles-albimanus.html" class="sf-depth-3"><i>Anopheles albimanus</i></a></li><li id="menu-3157-1" class="middle even sf-item-2 sf-depth-3 sf-no-children"><a href="organisms/anopheles-arabiensis.html" class="sf-depth-3"><i>Anopheles arabiensis</i></a></li><li id="menu-3342-1" class="middle odd sf-item-3 sf-depth-3 sf-no-children"><a href="organisms/anopheles-atroparvus.html" class="sf-depth-3"><i>Anopheles atroparvus</i></a></li><li id="menu-3158-1" class="middle even sf-item-4 sf-depth-3 sf-no-children"><a href="organisms/anopheles-christyi.html" title="" class="sf-depth-3"><i>Anopheles christyi</i></a></li><li id="menu-3687-1" class="middle odd sf-item-5 sf-depth-3 sf-no-children"><a href="organisms/anopheles-coluzzii.html" title="" class="sf-depth-3"><i>Anopheles coluzzii</i></a></li><li id="menu-3344-1" class="middle even sf-item-6 sf-depth-3 sf-no-children"><a href="organisms/anopheles-culicifacies.html" class="sf-depth-3"><i>Anopheles culicifacies</i></a></li><li id="menu-1781-1" class="middle odd sf-item-7 sf-depth-3 sf-no-children"><a href="organisms/anopheles-darlingi.html" title="" class="sf-depth-3"><i>Anopheles darlingi</i></a></li><li id="menu-3161-1" class="middle even sf-item-8 sf-depth-3 sf-no-children"><a href="organisms/anopheles-dirus.html" class="sf-depth-3"><i>Anopheles dirus</i></a></li><li id="menu-3159-1" class="middle odd sf-item-9 sf-depth-3 sf-no-children"><a href="organisms/anopheles-epiroticus.html" title="" class="sf-depth-3"><i>Anopheles epiroticus</i></a></li><li id="menu-3346-1" class="middle even sf-item-10 sf-depth-3 sf-no-children"><a href="organisms/anopheles-farauti.html" title="" class="sf-depth-3"><i>Anopheles farauti</i></a></li><li id="menu-3160-1" class="middle odd sf-item-11 sf-depth-3 sf-no-children"><a href="organisms/anopheles-funestus.html" class="sf-depth-3"><i>Anopheles funestus</i></a></li><li id="menu-776-1" class="middle even sf-item-12 sf-depth-3 sf-no-children"><a href="organisms/anopheles-gambiae.html" class="sf-depth-3"><i>Anopheles gambiae</i></a></li><li id="menu-3348-1" class="middle odd sf-item-13 sf-depth-3 sf-no-children"><a href="organisms/anopheles-maculatus.html" class="sf-depth-3"><i>Anopheles maculatus</i></a></li><li id="menu-3343-1" class="middle even sf-item-14 sf-depth-3 sf-no-children"><a href="organisms/anopheles-melas.html" class="sf-depth-3"><i>Anopheles melas</i></a></li><li id="menu-3345-1" class="middle odd sf-item-15 sf-depth-3 sf-no-children"><a href="organisms/anopheles-merus.html" class="sf-depth-3"><i>Anopheles merus</i></a></li><li id="menu-3163-1" class="middle even sf-item-16 sf-depth-3 sf-no-children"><a href="organisms/anopheles-minimus.html" class="sf-depth-3"><i>Anopheles minimus</i></a></li><li id="menu-3162-1" class="middle odd sf-item-17 sf-depth-3 sf-no-children"><a href="organisms/anopheles-quadriannulatus.html" class="sf-depth-3"><i>Anopheles quadriannulatus</i></a></li><li id="menu-3347-1" class="middle even sf-item-18 sf-depth-3 sf-no-children"><a href="organisms/anopheles-sinensis.html" class="sf-depth-3"><i>Anopheles sinensis</i></a></li><li id="menu-1780-1" class="last odd sf-item-19 sf-depth-3 sf-no-children"><a href="organisms/anopheles-stephensi.html" class="sf-depth-3"><i>Anopheles stephensi</i></a></li></ol></li></ul></li><li id="menu-2869-1" class="middle odd sf-item-3 sf-depth-2 sf-no-children"><a href="organisms/biomphalaria-glabrata.html" class="sf-depth-2"><i>Biomphalaria glabrata</i></a></li><li id="menu-7815-1" class="middle even sf-item-4 sf-depth-2 sf-no-children"><a href="organisms/cimex-lectularius.html" title="" class="sf-depth-2"><i>Cimex lectularius</i></a></li><li id="menu-974-1" class="middle odd sf-item-5 sf-depth-2 sf-no-children"><a href="organisms/culex-quinquefasciatus.html" title="" class="sf-depth-2"><i>Culex quinquefasciatus</i></a></li><li id="menu-4123-1" class="middle even sf-item-6 sf-depth-2 menuparent"><a href="taxonomy/glossina.html" title="" class="sf-depth-2 menuparent"><table class="menuarrow"><tr><td class="menuarrowtext"><i>Glossina</i> </td><td class="menuarrowicon"><img class="menuarrow" src="sites/default/files/ftp/triang.png" alt="submenu arrow"></td></tr></table></a><ul class="sf-megamenu"><li class="sf-megamenu-wrapper middle even sf-item-6 sf-depth-2 menuparent"><ol><li id="menu-4124-1" class="first odd sf-item-1 sf-depth-3 sf-no-children"><a href="organisms/glossina-austeni.html" title="Vector of Animal African Trypanosomiasis" class="sf-depth-3"><i>Glossina austeni</i></a></li><li id="menu-4125-1" class="middle even sf-item-2 sf-depth-3 sf-no-children"><a href="organisms/glossina-brevipalpis.html" title="Ancestral vector of Animal African Trypanosomiasis (AAT)" class="sf-depth-3"><i>Glossina brevipalpis</i></a></li><li id="menu-4126-1" class="middle odd sf-item-3 sf-depth-3 sf-no-children"><a href="organisms/glossina-fuscipes.html" title="Vector of Trypanosoma brucei rhodesiense in East Africa" class="sf-depth-3"><i>Glossina fuscipes</i></a></li><li id="menu-993-1" class="middle even sf-item-4 sf-depth-3 sf-no-children"><a href="organisms/glossina-morsitans.html" title="Vector of Human African Trypanosomiasis (sleeping sickness)" class="sf-depth-3"><i>Glossina morsitans</i></a></li><li id="menu-4127-1" class="middle odd sf-item-5 sf-depth-3 sf-no-children"><a href="organisms/glossina-pallidipes.html" title="Major vector of Trypanosoma brucei brucei, Minor vector of Trypanosoma brucei rhodesiense" class="sf-depth-3"><i>Glossina pallidipes</i></a></li><li id="menu-5190-1" class="last even sf-item-6 sf-depth-3 sf-no-children"><a href="organisms/glossina-palpalis.html" title="" class="sf-depth-3"><i>Glossina palpalis</i></a></li></ol></li></ul></li><li id="menu-971-1" class="middle odd sf-item-7 sf-depth-2 sf-no-children"><a href="organisms/ixodes-scapularis.html" title="" class="sf-depth-2"><i>Ixodes scapularis</i></a></li><li id="menu-15064-1" class="middle even sf-item-8 sf-depth-2 sf-no-children"><a href="organisms/leptotrombidium-deliense.html" title="" class="sf-depth-2"><i>Leptotrombidium deliense</i></a></li><li id="menu-2867-1" class="middle odd sf-item-9 sf-depth-2 sf-no-children"><a href="organisms/lutzomyia-longipalpis.html" title="" class="sf-depth-2"><i>Lutzomyia longipalpis</i></a></li><li id="menu-3850-1" class="middle even sf-item-10 sf-depth-2 sf-no-children"><a href="organisms/musca-domestica.html" title="Musca domestica" class="sf-depth-2"><i>Musca domestica</i></a></li><li id="menu-975-1" class="middle odd sf-item-11 sf-depth-2 sf-no-children"><a href="organisms/pediculus-humanus.html" title="" class="sf-depth-2"><i>Pediculus humanus</i></a></li><li id="menu-2868-1" class="middle even sf-item-12 sf-depth-2 sf-no-children"><a href="organisms/phlebotomus-papatasi.html" title="" class="sf-depth-2"><i>Phlebotomus papatasi</i></a></li><li id="menu-976-1" class="middle odd sf-item-13 sf-depth-2 sf-no-children"><a href="organisms/rhodnius-prolixus.html" title="" class="sf-depth-2"><i>Rhodnius prolixus</i></a></li><li id="menu-6654-1" class="middle even sf-item-14 sf-depth-2 sf-no-children"><a href="organisms/sarcoptes-scabiei-var-canis.html" class="sf-depth-2"><i>Sarcoptes scabiei</i> var. <i>canis</i></a></li><li id="menu-6138-1" class="middle odd sf-item-15 sf-depth-2 sf-no-children"><a href="organisms/stomoxys-calcitrans.html" title="Stomoxys calcitrans" class="sf-depth-2"><i>Stomoxys calcitrans</i></a></li><li id="menu-1786-1" class="last even sf-item-16 sf-depth-2 sf-no-children"><a href="organisms.html" title="" class="sf-depth-2">All organisms...</a></li></ul></li><li id="menu-988-1" class="middle even sf-item-4 sf-depth-1 menuparent"><a href="navigation/downloads.html" title="Download VectorBase data" class="sf-depth-1 menuparent">Downloads</a><ul><li id="menu-1775-1" class="first odd sf-item-1 sf-depth-2 sf-no-children"><a href="downloads.html" title="Raw data files stored by VectorBase." class="sf-depth-2">Data files</a></li><li id="menu-1336-1" class="middle even sf-item-2 sf-depth-2 sf-no-children"><a href="content/software-downloads.html" title="Open source software from VectorBase developers" class="sf-depth-2">Software</a></li><li id="menu-1397-1" class="middle odd sf-item-3 sf-depth-2 sf-no-children"><a href="image-gallery.html" title="" class="sf-depth-2">Images</a></li><li id="menu-1787-1" class="last even sf-item-4 sf-depth-2 sf-no-children"><a href="downloadsee7a.html" title="" class="sf-depth-2">Ontologies</a></li></ul></li><li id="menu-1002-1" class="middle odd sf-item-5 sf-depth-1 menuparent"><a href="navigation/tools.html" title="" class="sf-depth-1 menuparent">Tools</a><ul><li id="menu-1004-1" class="first odd sf-item-1 sf-depth-2 sf-no-children"><a href="blast.html" title="Sequence similarity searching" class="sf-depth-2">BLAST</a></li><li id="menu-1009-1" class="middle even sf-item-2 sf-depth-2 sf-no-children"><a href="clustalw.html" title="Align two or more sequences" class="sf-depth-2">ClustalW</a></li><li id="menu-1010-1" class="middle odd sf-item-3 sf-depth-2 sf-no-children"><a href="hmmer.html" title="Sensitive profile-based sequence search" class="sf-depth-2">HMMER</a></li><li id="menu-1008-1" class="last even sf-item-4 sf-depth-2 sf-no-children"><a href="ontology-browser.html" title="" class="sf-depth-2">Ontology Browser</a></li></ul></li><li id="menu-973-1" class="middle even sf-item-6 sf-depth-1 menuparent"><a href="navigation/data.html" title="" class="sf-depth-1 menuparent">Data</a><ul><li id="menu-3166-1" class="first odd sf-item-1 sf-depth-2 sf-no-children"><a href="genomes.html" class="sf-depth-2">Genomes</a></li><li id="menu-2270-1" class="middle even sf-item-2 sf-depth-2 sf-no-children"><a href="transcriptomes.html" class="sf-depth-2">Transcriptomes</a></li><li id="menu-9464-1" class="middle odd sf-item-3 sf-depth-2 sf-no-children"><a href="proteomes.html" class="sf-depth-2">Proteomes</a></li><li id="menu-1461-1" class="middle even sf-item-4 sf-depth-2 sf-no-children"><a href="mitochondrial-genomes.html" class="sf-depth-2">Mitochondrial sequences</a></li><li id="menu-4324-1" class="last odd sf-item-5 sf-depth-2 sf-no-children"><a href="releases.html" class="sf-depth-2">Release notes</a></li></ul></li><li id="menu-1001-1" class="active-trail middle odd sf-item-7 sf-depth-1 menuparent"><a href="navigation/help.html" title="" class="sf-depth-1 menuparent">Help</a><ul><li id="menu-970-1" class="first odd sf-item-1 sf-depth-2 sf-no-children"><a href="faqs.html" title="Frequently asked questions (with answers)" class="sf-depth-2">FAQs</a></li><li id="menu-4033-1" class="middle even sf-item-2 sf-depth-2 sf-no-children"><a href="tutorials.html" class="sf-depth-2">Tutorials</a></li><li id="menu-1631-1" class="active-trail middle odd sf-item-3 sf-depth-2 sf-no-children"><a href="glossary.html" class="sf-depth-2 active">Glossary</a></li><li id="menu-1334-1" class="middle even sf-item-4 sf-depth-2 sf-no-children"><a href="faqs/how-cite-vectorbase.html" title="" class="sf-depth-2">How to cite VectorBase</a></li><li id="menu-1616-1" class="middle odd sf-item-5 sf-depth-2 sf-no-children"><a href="data-policies.html" class="sf-depth-2">Data access and Submission policies</a></li><li id="menu-3845-1" class="last even sf-item-6 sf-depth-2 sf-no-children"><a href="solr-information.html" title="Description of indexed solr fields" class="sf-depth-2">SOLR Fields</a></li></ul></li><li id="menu-1013-1" class="middle even sf-item-8 sf-depth-1 menuparent"><a href="navigation/community.html" title="" class="sf-depth-1 menuparent">Community</a><ul><li id="menu-1777-1" class="first odd sf-item-1 sf-depth-2"><a href="content/submit-data.html" class="sf-depth-2">Submit Data</a><ul class="sf-megamenu"><li class="sf-megamenu-wrapper first odd sf-item-1 sf-depth-2"></li></ul></li><li id="menu-1016-1" class="middle even sf-item-2 sf-depth-2 sf-no-children"><a href="mailing-lists.html" title="" class="sf-depth-2">Mailing lists</a></li><li id="menu-1608-1" class="middle odd sf-item-3 sf-depth-2 sf-no-children"><a href="publications.html" class="sf-depth-2">Publications</a></li><li id="menu-10744-1" class="middle even sf-item-4 sf-depth-2 sf-no-children"><a href="workshops.html" class="sf-depth-2">Workshops/Meetings</a></li><li id="menu-12943-1" class="middle odd sf-item-5 sf-depth-2 sf-no-children"><a href="jobpostings.html" class="sf-depth-2">Job postings</a></li><li id="menu-4034-1" class="middle even sf-item-6 sf-depth-2 sf-no-children"><a href="news.html" class="sf-depth-2">News</a></li><li id="menu-4035-1" class="middle odd sf-item-7 sf-depth-2 sf-no-children"><a href="newsletters.html" class="sf-depth-2">Newsletters</a></li><li id="menu-2269-1" class="middle even sf-item-8 sf-depth-2 sf-no-children"><a href="useful-links.html" title="" class="sf-depth-2">Useful links</a></li><li id="menu-1014-1" class="middle odd sf-item-9 sf-depth-2 sf-no-children"><a href="http://www.facebook.com/pages/VectorBase/189608142260" title="" class="sf-depth-2"><table class="menuarrow"><tr><td class="menuarrowtext">Facebook  </td><td class="menuarrowicon"><img src="sites/default/files/ftp/fb24.png" alt="link icon"></td></tr></table></a></li><li id="menu-1015-1" class="last even sf-item-10 sf-depth-2 sf-no-children"><a href="http://twitter.com/VectorBase" title="" class="sf-depth-2"><table class="menuarrow"><tr><td class="menuarrowtext">Twitter  </td><td class="menuarrowicon"><img src="sites/default/files/ftp/tw24.png" alt="link icon"></td></tr></table></a></li></ul></li><li id="menu-994-1" class="last odd sf-item-9 sf-depth-1 sf-no-children"><a href="contact.html" title="Send us comments, suggestions, or questions" class="sf-depth-1">Contact Us</a></li></ul>      </div>
      </div><!-- /inner-inner -->
	  </div><!-- /inner-wrapper -->
  </div><!-- /block-inner -->
</div><!-- /block -->    </div><!-- /main-menu-inner -->
  </div><!-- /main-menu -->
</div><!-- /main-menu-wrapper -->
        </div><!-- /header-group -->
      </div><!-- /header-group-wrapper -->

      <div id="preface-top-outer"> </div>

      <!-- main region: width = grid_width -->
      <div id="main-wrapper" class="main-wrapper full-width clearfix">
        <div id="main" class="main region grid16-16">
          <div id="main-inner" class="main-inner inner clearfix">
            
            <!-- main group: width = grid_width - sidebar_first_width -->
            <div id="main-group" class="main-group region nested grid16-16">
              <div id="main-group-inner" class="main-group-inner inner">
                 

                <div id="main-content" class="main-content region nested">
                  <div id="main-content-inner" class="main-content-inner inner">
                    <!-- content group: width = grid_width - sidebar_first_width - sidebar_second_width -->
                    <div id="content-group" class="content-group region nested grid16-16">
                      <div id="content-group-inner" style="background-color:white;" class="content-group-inner inner">
                        <div id="breadcrumbs" class="breadcrumbs block">
<div id="breadcrumbs-inner" class="breadcrumbs-inner gutter">
<a href="index.html">Home</a> &raquo; <a href="navigation/help.html" title="">Help</a> &raquo; Glossary
</div><!-- /breadcrumbs-inner -->
</div><!-- /breadcrumbs -->
                        
                        <div id="content-region" class="content-region region nested">
                          <div id="content-region-inner" class="content-region-inner inner">

                            <a name="main-content-area" id="main-content-area"></a>

                                                        
                              <!--DKDK highlighted -->

                                                                                    <h1 class="title gutter">Glossary</h1>
                                                                                                                                              
<!-- content region -->
  <div id="content" class="region region-content content nested grid16-16">
    <div id="content-inner" class="content-inner inner">
      <div id="block-system-main" class="block block-system first  odd">
  <div class="inner clearfix gutter">
  	<div class="inner-wrapper">
      <div class="inner-inner">
                        <div class="content clearfix">
        <div class="lexicon-admin-links"></div><div id="glossary">
      <div class="lexicon-description">
      Glossary of terms for the Glossary module    </div>
      <div class="lexicon-links">
  <a href="glossary.html#letter_a" class="lexicon-item active">a</a> | <a href="glossary.html#letter_b" class="lexicon-item active">b</a> | <a href="glossary.html#letter_c" class="lexicon-item active">c</a> | <a href="glossary.html#letter_d" class="lexicon-item active">d</a> | <a href="glossary.html#letter_e" class="lexicon-item active">e</a> | <a href="glossary.html#letter_f" class="lexicon-item active">f</a> | <a href="glossary.html#letter_g" class="lexicon-item active">g</a> | <a href="glossary.html#letter_h" class="lexicon-item active">h</a> | <a href="glossary.html#letter_i" class="lexicon-item active">i</a> | <a href="glossary.html#letter_j" class="lexicon-item active">j</a> | <a href="glossary.html#letter_k" class="lexicon-item active">k</a> | <a href="glossary.html#letter_l" class="lexicon-item active">l</a> | <a href="glossary.html#letter_m" class="lexicon-item active">m</a> | <a href="glossary.html#letter_n" class="lexicon-item active">n</a> | <a href="glossary.html#letter_o" class="lexicon-item active">o</a> | <a href="glossary.html#letter_p" class="lexicon-item active">p</a> | <a href="glossary.html#letter_q" class="lexicon-item active">q</a> | <a href="glossary.html#letter_r" class="lexicon-item active">r</a> | <a href="glossary.html#letter_s" class="lexicon-item active">s</a> | <a href="glossary.html#letter_t" class="lexicon-item active">t</a> | <a href="glossary.html#letter_u" class="lexicon-item active">u</a> | <a href="glossary.html#letter_v" class="lexicon-item active">v</a> | <a href="glossary.html#letter_y" class="lexicon-item active">y</a> | <a href="glossary.html#letter_z" class="lexicon-item active">z</a></div>
<div class="lexicon-alphabar-instructions">
  Click one of the letters above to advance the page to terms beginning with that letter.</div>
  <div class="lexicon-list">
            <a id="letter_a"></a>
  <h2 class="lexicon-letter">A</h2>
  <dl>
    <dt>
<a id="Accession_number"></a>
  <a href="glossary/accession-number.html">Accession number</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Accession number%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A unique identifier given to a sequence, sample or other item when it is submitted to a repository (for example GenBank).</p>
         </dd>
<dt>
<a id="AGP_file_A_Golden_Path"></a>
  <a href="glossary/agp-file-golden-path.html">AGP file (A Golden Path)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522AGP file (A Golden Path)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A file provided to VectorBase that describes how the longer sequences in the genome <a href="glossary.html#Assembly" title="When the genome or transcriptome of a species is sequenced, typically many short random fragments are sequenced and reassembled by a computer algorithm into longer contiguous sequences called contigs. Genomic contigs may be assembled into longer sequences called scaffolds and sometimes, if the depth of sequencing is high enough, there may be enough information to assemble most of the scaffolds into chromosomes." class="lexicon-term active">assembly</a> were assembled from shorter sequences. For example, an AGP file can describe how a chromosome is assembled from a collection of scaffolds or a collection of contigs. For an AGP file that describes how a <a href="glossary.html#Scaffold" title="Supercontigs or scaffolds are sets of ordered, oriented contigs. They are longer sequences than contigs, but shorter than full chromosomes." class="lexicon-term active">scaffold</a> is assembled from a collection of contigs, each <a href="glossary.html#Contig" title="A contig is a contiguous stretch of DNA sequence without gaps that has been assembled solely based on direct sequencing information.    Short sequences (reads) from a fragmented genome are compared against one another, and overlapping reads are merged to produce one long sequence. This merging process is iterative: overlapping reads are added to the merged sequence whenever possible and so the merged sequence becomes even longer. When no further reads overlap the long merged sequence, then this sequence - called a contig - has reached its maximum length.    Contig can be used in other contexts: A contig can be the sequence corresponding to only one clone. A contig map shows the regions of a chromosome where contiguous DNA segments overlap." class="lexicon-term active">contig</a> will be listed on a separate line in the AGP file and the line will include information about where the contig lies within the scaffold and the orientation of the contig.</p>
<ul><li><a href="https://www.vectorbase.org/downloads?field_organism_taxonomy_tid=All&amp;field_download_file_type_tid=All&amp;field_download_file_format_tid=1169&amp;field_status_value=Current">AGPs available from VectorBase</a></li>
</ul>         </dd>
<dt>
<a id="Algorithm"></a>
  <a href="glossary/algorithm.html">Algorithm</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Algorithm%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A sequence of computational tasks or actions that carry out a specific function.</p>
         </dd>
<dt>
<a id="Alignment"></a>
  <a href="glossary/alignment.html">Alignment</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Alignment%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A comparison between two or more sequences by matching identical and/or similar residues and assigning a score to the match.</p>
         </dd>
<dt>
<a id="Allele"></a>
  <a href="glossary/allele.html">Allele</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Allele%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>An <a href="glossary.html#Allele" title="An allele is an alternative form of a nucleotide sequence, a gene or a locus in the genome. The term was originally used to describe variation among protein coding genes, but it also refers to variation among non-coding genes or DNA sequences." class="lexicon-term active">allele</a> is an alternative form of a nucleotide sequence, a gene or a locus in the genome. The term was originally used to describe variation among <a href="glossary.html#Protein_coding" title="A protein coding transcript is a spliced mRNA that leads to a protein product." class="lexicon-term active">protein coding</a> genes, but it also refers to variation among non-coding genes or DNA sequences.</p>
         </dd>
<dt>
<a id="Allele_frequency"></a>
  <a href="glossary/allele-frequency.html">Allele frequency</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Allele frequency%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A measure of how prevalent an <a href="glossary.html#Allele" title="An allele is an alternative form of a nucleotide sequence, a gene or a locus in the genome. The term was originally used to describe variation among protein coding genes, but it also refers to variation among non-coding genes or DNA sequences." class="lexicon-term active">allele</a> or <a href="glossary.html#Genotype" title="Specific alleles present in an individual&#039;s genome, or the genetic makeup of one organism." class="lexicon-term active">genotype</a> is in a population. In Ensembl, it is displayed ranging from 0 (zero) to 1 (one).</p>
         </dd>
<dt>
<a id="Alternate_sequence"></a>
  <a href="glossary/alternate-sequence.html">Alternate sequence</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Alternate sequence%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Genomic sequence that differs from the genomic DNA on the primary <a href="glossary.html#Assembly" title="When the genome or transcriptome of a species is sequenced, typically many short random fragments are sequenced and reassembled by a computer algorithm into longer contiguous sequences called contigs. Genomic contigs may be assembled into longer sequences called scaffolds and sometimes, if the depth of sequencing is high enough, there may be enough information to assemble most of the scaffolds into chromosomes." class="lexicon-term active">assembly</a>. The alternate sequences come in two types: allelic sequence (<a href="glossary.html#Haplotypes" title="A set of genes or markers on one chromosome that are inherited together. Often refers to SNPs that are closely linked (i.e. have a high linkage disequilibrium (LD) value, and are inherited together.) In Region in Detail, the haplotype regions are coloured with a red background." class="lexicon-term active">haplotypes</a> and novel patches) and fix patches. Novel patches represent new allelic loci but they are not necessarily haplotypes. Fix patches are where the primary assembly was found to be incorrect, and the patch reflects the corrected sequence. Both haplotypes, novel patches and fix patches are determined by the GRC, not by Ensembl. When using the API, the primary assembly is referred to as reference sequence and <a href="glossary.html#Alternate_sequence" title="Genomic sequence that differs from the genomic DNA on the primary assembly. The alternate sequences come in two types: allelic sequence (haplotypes and novel patches) and fix patches. Novel patches represent new allelic loci but they are not necessarily haplotypes. Fix patches are where the primary assembly was found to be incorrect, and the patch reflects the corrected sequence. Both haplotypes, novel patches and fix patches are determined by the GRC, not by Ensembl. When using the API, the primary assembly is referred to as reference sequence and alternate sequence is referred to as non-reference sequence." class="lexicon-term active">alternate sequence</a> is referred to as non-reference sequence.</p>
         </dd>
<dt>
<a id="ambiguity_code"></a>
  <a href="glossary/ambiguity-code.html">ambiguity code</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522ambiguity code%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>The standard ambiguity codes for nucleotides are provided by IUPAC (INTERNATIONAL UNION OF PURE AND APPLIED CHEMISTRY) and indicate the possible nucleotides that can occur at a given position. The symbols are valid for both DNA and RNA and are shown below:    A = adenine    C = cytosine    G = guanine    T = thymine    R = G A (purine)    Y = T C (pyrimidine)    K = G T (keto)    M = A C (amino)    S = G C (strong bonds)    W = A T (weak bonds)    B = G T C (all but A)    D = G A T (all but C)    H = A C T (all but G)    V = G C A (all but T)    N = A G C T (any)</p>
         </dd>
<dt>
<a id="Ambiguous_ORF"></a>
  <a href="glossary/ambiguous-orf.html">Ambiguous ORF</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Ambiguous ORF%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Ambiguous Open Reading Frame. A non-coding <a href="glossary.html#Transcript" title="Nucleotide sequence resulting from the transcription of the genomic DNA to mRNA. One gene can have different transcripts or splice variants resulting from the alternative splicing of different exons in genes." class="lexicon-term active">transcript</a> believed to be <a href="glossary.html#Protein_coding" title="A protein coding transcript is a spliced mRNA that leads to a protein product." class="lexicon-term active">protein coding</a>, with more than one possible ORF.</p>
         </dd>
<dt>
<a id="Anthropophilic"></a>
  <a href="taxonomy/term/1366.html">Anthropophilic</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Anthropophilic%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p><a href="glossary.html#Anthropophilic" title="Anthropophilic mosquitoes preferentially feed on humans." class="lexicon-term active">Anthropophilic</a> mosquitoes preferentially feed on humans.</p>
         </dd>
<dt>
<a id="Antisense"></a>
  <a href="glossary/antisense.html">Antisense</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Antisense%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Non-coding <a href="glossary.html#Transcript" title="Nucleotide sequence resulting from the transcription of the genomic DNA to mRNA. One gene can have different transcripts or splice variants resulting from the alternative splicing of different exons in genes." class="lexicon-term active">transcript</a> believed to be an <a href="glossary.html#Antisense" title="Non-coding transcript believed to be an antisense product used in the regulation of the gene to which it belongs." class="lexicon-term active">antisense</a> product used in the regulation of the gene to which it belongs.</p>
         </dd>
<dt>
<a id="API_Application_Programming_Interface"></a>
  <a href="glossary/api-application-programming-interface.html">API (Application Programming Interface)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522API (Application Programming Interface)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A series of routines that applications can use to make the operating system request and carry out lower-level services.</p>
         </dd>
<dt>
<a id="Artifact_in_the_context_of_a_transcript"></a>
  <a href="glossary/artifact-context-transcript.html">Artifact ((in the context of a transcript))</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Artifact ((in the context of a transcript))%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Error in the sequence in a public database (for example UniProtKB, NCBI <a href="glossary.html#RefSeq" title="NCBI&#039;s Reference Sequences (RefSeq) database is a curated database of Genbank&#039;s genomes, mRNAs and proteins. RefSeq attempts to provide a comprehensive, integrated, non-redundant set of sequences, including genomic DNA, tRNA, and protein products, providing a stable reference for gene identification and characterization, mutation analysis, expression studies, polymorphism discovery, and comparative analyses." class="lexicon-term active">RefSeq</a>). Annotation is by the VEGA/Havana project.</p>
         </dd>
<dt>
<a id="Assembly"></a>
  <a href="glossary/assembly.html">Assembly</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Assembly%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>When the genome or transcriptome of a species is sequenced, typically many short random fragments are sequenced and reassembled by a computer <a href="glossary.html#Algorithm" title="A sequence of computational tasks or actions that carry out a specific function." class="lexicon-term active">algorithm</a> into longer contiguous sequences called contigs. Genomic contigs may be assembled into longer sequences called scaffolds and sometimes, if the depth of sequencing is high enough, there may be enough information to assemble most of the scaffolds into chromosomes.</p>
         </dd>
<dt>
<a id="ATV_A_Tree_Viewer"></a>
  <a href="glossary/atv-tree-viewer.html">ATV (A Tree Viewer)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522ATV (A Tree Viewer)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>An application (Java tool) for the visualisation of phylogenetic trees. Allows the possibility to edit and export data. See Zmasek et al.</p>
         </dd>
  </dl>
                  <a id="letter_b"></a>
  <h2 class="lexicon-letter">B</h2>
  <dl>
    <dt>
<a id="BAC_Bacterial_Artificial_Chromosome"></a>
  <a href="glossary/bac-bacterial-artificial-chromosome.html">BAC (Bacterial Artificial Chromosome)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522BAC (Bacterial Artificial Chromosome)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A vector used to <a href="glossary.html#Clone" title="A segment of DNA that has been inserted into a vector molecule, such as a plasmid, and then replicated to form many identical copies." class="lexicon-term active">clone</a> DNA fragments (100 to 300-kb insert size; average, 150 kb) from another species so that it can be replicated in bacteria.</p>
         </dd>
<dt>
<a id="Base_pairs_number_of_base_pairs_in_the_genome"></a>
  <a href="glossary/base-pairs-number-base-pairs-genome.html">Base pairs (number of base pairs in the genome)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Base pairs (number of base pairs in the genome)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>The base pairs length on pages such as the whole genome display (next to the <a href="glossary.html#Golden_path_length" title="The golden path is the length of the reference assembly. It consists of the sum of all top-level sequences in the seq_region table, omitting any redundant regions such as haplotypes and PARs (pseudoautosomal regions)." class="lexicon-term active">golden path length</a>) is based on the assembled end position of the last seq_region in each chromosome (from the AGP file), or if there is a terminal gap it is set to the assembled end location of that terminal gap.</p>
         </dd>
<dt>
<a id="Biotype"></a>
  <a href="glossary/biotype.html">Biotype</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Biotype%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A gene or <a href="glossary.html#Transcript" title="Nucleotide sequence resulting from the transcription of the genomic DNA to mRNA. One gene can have different transcripts or splice variants resulting from the alternative splicing of different exons in genes." class="lexicon-term active">transcript</a> classification. Transcript types include <a href="glossary.html#Protein_coding" title="A protein coding transcript is a spliced mRNA that leads to a protein product." class="lexicon-term active">protein coding</a>, <a href="glossary.html#Pseudogene" title="A noncoding sequence similar to an active protein. A sequence that closely resembles a known functional gene, at another locus within a genome, that is non-functional as a consequence of (usually several) mutations that prevent either its transcription or translation (or both). In general, pseudogenes result from either reverse transcription of a transcript of their &quot;normal&quot; paralog (SO:0000043) (in which case the pseudogene typically lacks introns and includes a poly(A) tail) or from recombination (SO:0000044) (in which case the pseudogene is typically a tandem duplication of its &quot;normal&quot; paralog). Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000336" class="lexicon-term active">pseudogene</a>, and non-coding RNAs.</p>
         </dd>
<dt>
<a id="BLAST"></a>
  <a href="glossary/blast.html">BLAST</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522BLAST%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p><a href="glossary.html#BLAST" title="BLAST (Basic Local Alignment Search Tool) is a sequence comparison algorithm optimised for speed which is used to search sequence databases for optimal local alignments to a query. (Altschul et al., J Mol Biol 215:403-410; 1990)" class="lexicon-term active">BLAST</a> (Basic Local <a href="glossary.html#Alignment" title="A comparison between two or more sequences by matching identical and/or similar residues and assigning a score to the match." class="lexicon-term active">Alignment</a> Search Tool) is a sequence comparison <a href="glossary.html#Algorithm" title="A sequence of computational tasks or actions that carry out a specific function." class="lexicon-term active">algorithm</a> optimised for speed which is used to search sequence databases for optimal local alignments to a query. (Altschul et al., J Mol Biol 215:403-410; 1990)</p>
         </dd>
<dt>
<a id="BLAT_BLAST-Like_Alignment_Tool"></a>
  <a href="glossary/blat-blast-alignment-tool.html">BLAT (BLAST-Like Alignment Tool)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522BLAT (BLAST-Like Alignment Tool)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>An mRNA/DNA and cross-species protein sequence analysis tool to quickly find sequences of 95% and greater <a href="glossary.html#Similarity" title="How well one sequence matches another determined by calculation by an alignment program of identical and conserved residues." class="lexicon-term active">similarity</a> of length 40 bases or more. (Kent, W.J. 2002. BLAT -- The <a href="glossary.html#BLAST" title="BLAST (Basic Local Alignment Search Tool) is a sequence comparison algorithm optimised for speed which is used to search sequence databases for optimal local alignments to a query. (Altschul et al., J Mol Biol 215:403-410; 1990)" class="lexicon-term active">BLAST</a>-Like <a href="glossary.html#Alignment" title="A comparison between two or more sequences by matching identical and/or similar residues and assigning a score to the match." class="lexicon-term active">Alignment</a> Tool. Genome Research 4: 656-664)</p>
         </dd>
<dt>
<a id="BLOSUM_62_Blocks_Substitution_Matrix"></a>
  <a href="glossary/blosum-62-blocks-substitution-matrix.html">BLOSUM 62 (Blocks Substitution Matrix)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522BLOSUM 62 (Blocks Substitution Matrix)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A matrix that defines scores for amino acid substitutions, reflecting the <a href="glossary.html#Similarity" title="How well one sequence matches another determined by calculation by an alignment program of identical and conserved residues." class="lexicon-term active">similarity</a> of physicochemical properties, and observed substitution frequencies. The BLOSUM 62 matrix is tailored using sequences sharing no more than 62% identity (sequences closer evolutionary, were represented by a single sequence in the <a href="glossary.html#Alignment" title="A comparison between two or more sequences by matching identical and/or similar residues and assigning a score to the match." class="lexicon-term active">alignment</a> to avoid bias from using related family members). (Henikoff and Henikoff, Proc Natl Acad Sci U S A 89:10915-10919; 1992).</p>
         </dd>
  </dl>
                  <a id="letter_c"></a>
  <h2 class="lexicon-letter">C</h2>
  <dl>
    <dt>
<a id="Canonical_transcript"></a>
  <a href="glossary/canonical-transcript.html">Canonical transcript</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Canonical transcript%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>For human, the <a href="glossary.html#Canonical_transcript" title="For human, the canonical transcript for a gene is set according to the following hierarchy: 1. Longest CCDS translation with no stop codons. 2. If no (1), choose the longest Ensembl/Havana merged translation with no stop codons. 3. If no (2), choose the longest translation with no stop codons. 4. If no translation, choose the longest non-protein-coding transcript." class="lexicon-term active">canonical transcript</a> for a gene is set according to the following hierarchy: 1. Longest CCDS translation with no stop codons. 2. If no (1), choose the longest Ensembl/Havana merged translation with no stop codons. 3. If no (2), choose the longest translation with no stop codons. 4. If no translation, choose the longest non-protein-coding transcript.</p>
         </dd>
<dt>
<a id="cDNA_Complementary_DNA"></a>
  <a href="glossary/cdna-complementary-dna.html">cDNA (Complementary DNA)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522cDNA (Complementary DNA)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>DNA obtained by reverse transcription of a mRNA template. In bioinformatics jargon, cDNA is thought of as a DNA version of the mRNA sequence. Generally, cDNAs are denoted in coding or 'sense' orientation.</p>
         </dd>
<dt>
<a id="CDS_Coding_sequence"></a>
  <a href="glossary/cds-coding-sequence.html">CDS (Coding sequence)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522CDS (Coding sequence)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>The portion of a gene or an mRNA that codes for a protein. Introns are not coding sequences, nor are the 5' or 3' UTR. The coding sequence in a cDNA or mature mRNA includes everything from the start <a href="glossary.html#codon" title="Three base pairs in either DNA or RNA that code for an amino acid (or stop translation)." class="lexicon-term active">codon</a> through to the stop codon, inclusive.</p>
         </dd>
<dt>
<a id="Centimorgan_cM"></a>
  <a href="glossary/centimorgan-cm.html">Centimorgan (cM)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Centimorgan (cM)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A unit of genetic distance, determined by how frequently two genes on the same chromosome are inherited together. One centimorgan equals 1% recombinant offspring. In humans, 1 cM is about 1 x 10^6 bp</p>
         </dd>
<dt>
<a id="Chr:bp"></a>
  <a href="glossary/chrbp.html">Chr:bp</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Chr%3Abp%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>The chromosome location and coordinates in base pairs.</p>
         </dd>
<dt>
<a id="CIGAR_Compact_Idiosyncratic_Gapped_Alignment_Report"></a>
  <a href="glossary/cigar-compact-idiosyncratic-gapped-alignment-report.html">CIGAR (Compact Idiosyncratic Gapped Alignment Report)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522CIGAR (Compact Idiosyncratic Gapped Alignment Report)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Defines the sequence of matches/mismatches and deletions (or gaps). The cigar line defines the sequence of matches/mismatches and deletions (or gaps). For example, this cigar line 2MD3M2D2M will mean that the <a href="glossary.html#Alignment" title="A comparison between two or more sequences by matching identical and/or similar residues and assigning a score to the match." class="lexicon-term active">alignment</a> contains 2 matches/mismatches, 1 deletion (number 1 is omitted in order to save some space), 3 matches/mismatches, 2 deletions and 2 matches/mismatches. If the original sequence is:    Original sequence: AACGCTT    The aligned sequence will be:    cigar line: 2MD3M2D2M M M D M M M D D M M    A A - C G C - - T T</p>
         </dd>
<dt>
<a id="Clone"></a>
  <a href="glossary/clone.html">Clone</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Clone%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A segment of DNA that has been inserted into a vector molecule, such as a plasmid, and then replicated to form many identical copies.</p>
         </dd>
<dt>
<a id="CNV"></a>
  <a href="glossary/cnv.html">CNV</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522CNV%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Copy number variation. It is defined by SO (sequence ontology) as a variation that increases or decreases the copy number of a given region. See more details here.</p>
         </dd>
<dt>
<a id="codon"></a>
  <a href="glossary/codon.html">codon</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522codon%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Three base pairs in either DNA or RNA that code for an amino acid (or stop translation).</p>
         </dd>
<dt>
<a id="Contig"></a>
  <a href="glossary/contig.html">Contig</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Contig%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A <a href="glossary.html#Contig" title="A contig is a contiguous stretch of DNA sequence without gaps that has been assembled solely based on direct sequencing information.    Short sequences (reads) from a fragmented genome are compared against one another, and overlapping reads are merged to produce one long sequence. This merging process is iterative: overlapping reads are added to the merged sequence whenever possible and so the merged sequence becomes even longer. When no further reads overlap the long merged sequence, then this sequence - called a contig - has reached its maximum length.    Contig can be used in other contexts: A contig can be the sequence corresponding to only one clone. A contig map shows the regions of a chromosome where contiguous DNA segments overlap." class="lexicon-term active">contig</a> is a contiguous stretch of DNA sequence without gaps that has been assembled solely based on direct sequencing information.    Short sequences (reads) from a fragmented genome are compared against one another, and overlapping reads are merged to produce one long sequence. This merging process is iterative: overlapping reads are added to the merged sequence whenever possible and so the merged sequence becomes even longer. When no further reads overlap the long merged sequence, then this sequence - called a contig - has reached its maximum length.    Contig can be used in other contexts: A contig can be the sequence corresponding to only one <a href="glossary.html#Clone" title="A segment of DNA that has been inserted into a vector molecule, such as a plasmid, and then replicated to form many identical copies." class="lexicon-term active">clone</a>. A contig map shows the regions of a chromosome where contiguous DNA segments overlap.</p>
         </dd>
<dt>
<a id="Coordinate_system"></a>
  <a href="glossary/coordinate-system.html">Coordinate system</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Coordinate system%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>In VectorBase, the term "<a href="glossary.html#Coordinate_system" title="In VectorBase, the term &quot;coordinate system&quot; or &quot;coord_system&quot; identifies which level of the assembly we are working on. A genome assembly imported into VectorBase has up to three coordinate systems defined in the coord_system table: contigs, scaffolds or chromosomes.    We define one additional coordinate system: toplevel. Toplevel sequences are tagged in the seq_region_attrib table. Most gene annotation is done on toplevel sequence." class="lexicon-term active">coordinate system</a>" or "coord_system" identifies which level of the <a href="glossary.html#Assembly" title="When the genome or transcriptome of a species is sequenced, typically many short random fragments are sequenced and reassembled by a computer algorithm into longer contiguous sequences called contigs. Genomic contigs may be assembled into longer sequences called scaffolds and sometimes, if the depth of sequencing is high enough, there may be enough information to assemble most of the scaffolds into chromosomes." class="lexicon-term active">assembly</a> we are working on. A genome assembly imported into VectorBase has up to three coordinate systems defined in the coord_system table: contigs, scaffolds or chromosomes.<br /><br />    We define one additional coordinate system: <a href="glossary.html#Toplevel" title="The largest continuous sequence for an organism. The official technical definition for toplevel sequences are &#039;sequence regions in the genome assembly that are not a component of another sequence region&#039;. For example, when a genome is assembled into chromosomes, toplevel sequences will be chromosomes and unplaced scaffolds. If a genome has only been assembled into scaffolds, then toplevel sequences are scaffolds and unplaced contigs." class="lexicon-term active">toplevel</a>. Toplevel sequences are tagged in the seq_region_attrib table. Most gene annotation is done on toplevel sequence.</p>
         </dd>
<dt>
<a id="Cosmid"></a>
  <a href="glossary/cosmid.html">Cosmid</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Cosmid%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>DNA from a bacterial virus spliced with a small fragment of a genome (up to 50 kb) to be amplified and sequenced.</p>
         </dd>
<dt>
<a id="cytogenetic_map"></a>
  <a href="glossary/cytogenetic-map.html">cytogenetic map</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522cytogenetic map%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A banding pattern on a chromosome resulting from staining and examination by microscopy. Cytogenetic abnormalities such as deletions or inverted nucleotide sequences may be detected by examining and comparing banding patterns.</p>
         </dd>
  </dl>
                  <a id="letter_d"></a>
  <h2 class="lexicon-letter">D</h2>
  <dl>
    <dt>
<a id="DAS_Distributed_Annotation_System"></a>
  <a href="glossary/das-distributed-annotation-system.html">DAS (Distributed Annotation System)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522DAS (Distributed Annotation System)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A protocol for requesting and returning annotation data for genomic regions. See the BioDAS site for more information.</p>
         </dd>
<dt>
<a id="dbSNP"></a>
  <a href="glossary/dbsnp.html">dbSNP</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522dbSNP%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>The Single Nucleotide Polymorphism database (<a href="glossary.html#dbSNP" title="    The Single Nucleotide Polymorphism database (dbSNP) is a public-domain archive for a broad collection of simple (short) genetic polymorphisms. This collection of polymorphisms is maintained by NCBI and includes single-base nucleotide substitutions (also known as single nucleotide polymorphisms or SNPs), small-scale multi-base deletions or insertions (also called deletion insertion polymorphisms, indels or DIPs), and retroposable element insertions and microsatellite repeat variations (also called short tandem repeats or STRs). See more details here." class="lexicon-term active">dbSNP</a>) is a public-<a href="glossary.html#Domain" title="A region of special biological interest within a single protein sequence. However, a domain may also be defined as a region within the three-dimensional structure of a protein that may encompass regions of several distinct protein sequences that accomplishes a specific function. A domain class is a group of domains that share a common set of well-defined properties or characteristics." class="lexicon-term active">domain</a> archive for a broad collection of simple (short) genetic polymorphisms. This collection of polymorphisms is maintained by NCBI and includes single-base nucleotide substitutions (also known as single nucleotide polymorphisms or SNPs), small-scale multi-base deletions or insertions (also called deletion insertion polymorphisms, indels or DIPs), and retroposable element insertions and <a href="glossary.html#Microsatellite" title="A region in the genomic sequence containing short tandem repeats." class="lexicon-term active">microsatellite</a> <a href="glossary.html#repeat" title="Repetitive DNA in which the same sequence occurs multiple times." class="lexicon-term active">repeat</a> variations (also called short <a href="glossary.html#tandem_repeats" title="Multiple copies of the same base sequence on a chromosome; used as markers in physical mapping." class="lexicon-term active">tandem repeats</a> or STRs). See more details here.</p>
         </dd>
<dt>
<a id="dbVAR"></a>
  <a href="glossary/dbvar.html">dbVAR</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522dbVAR%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Database of genomic structural variation (<a href="glossary.html#SV" title="Structural variation. It is generally defined as a region of DNA approximately 1 kb and larger in size and can include inversions and balanced translocations or genomic imbalances (insertions and deletions), commonly referred to as copy number variants (CNVs). More details here." class="lexicon-term active">SV</a>), such as copy number variation. See the glossary term for SV. More details here.</p>
         </dd>
<dt>
<a id="DDBJ_DNA_Data_Bank_of_Japan"></a>
  <a href="glossary/ddbj-dna-data-bank-japan.html">DDBJ (DNA Data Bank of Japan)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522DDBJ (DNA Data Bank of Japan)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>DDBJ is the sole DNA data bank in Japan, which is officially certified to collect DNA sequences from researchers and to issue the internationally recognized <a href="glossary.html#Accession_number" title="A unique identifier given to a sequence, sample or other item when it is submitted to a repository (for example GenBank)." class="lexicon-term active">accession number</a> to data submitters. Data is exchanged with EMBL/EBI and GenBank/NCBI on a daily basis, and the three data banks share virtually the same data at any given time.</p>
         </dd>
<dt>
<a id="Disrupted_domain_in_the_context_of_a_transcript"></a>
  <a href="glossary/disrupted-domain-context-transcript.html">Disrupted domain ((in the context of a transcript))</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Disrupted domain ((in the context of a transcript))%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Coding region omiitted due to a splice variation. Annotation is by the VEGA/Havana project.</p>
         </dd>
<dt>
<a id="Domain"></a>
  <a href="glossary/domain.html">Domain</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Domain%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A region of special biological interest within a single protein sequence. However, a <a href="glossary.html#Domain" title="A region of special biological interest within a single protein sequence. However, a domain may also be defined as a region within the three-dimensional structure of a protein that may encompass regions of several distinct protein sequences that accomplishes a specific function. A domain class is a group of domains that share a common set of well-defined properties or characteristics." class="lexicon-term active">domain</a> may also be defined as a region within the three-dimensional structure of a protein that may encompass regions of several distinct protein sequences that accomplishes a specific function. A domain class is a group of domains that share a common set of well-defined properties or characteristics.</p>
         </dd>
<dt>
<a id="Dotter"></a>
  <a href="glossary/dotter.html">Dotter</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Dotter%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Ensembl DotterView is based on the program <a href="glossary.html#Dotter" title="Ensembl DotterView is based on the program Dotter, a dot-matrix program with dynamic threshold control suited for genomic DNA and protein sequence analysis. The Dotter tool provides a visual display of the sequence alignment it represents. The dotplot displays detailed comparison of two sequences. Every residue in one sequence is compared to every residue in the other sequence. The first sequence runs along the x-axis and the second sequence along the y-axis. In regions where the two sequences are similar to each other, a row of high scores will run diagonally across the dot matrix. If you&#039;re comparing a sequence against itself to find internal repeats, you&#039;ll notice that the main diagonal scores maximally, since it&#039;s the 100% perfect self-match. To make the score matrix more intelligible, the pairwise scores are averaged over a sliding window that runs diagonally. The averaged score matrix forms a three-dimensional landscape, with the two sequences in two dimensions and the height of the peaks in the third. This landscape is projected onto two dimensions by aid of grayscales - higher peaks are indicated by darker grays. Dotter was written by Erik L.L. Sonnhammer and Richard Durbin Gene 167: GC1-10 (1995)" class="lexicon-term active">Dotter</a>, a dot-matrix program with dynamic threshold control suited for genomic DNA and protein sequence analysis. The Dotter tool provides a visual display of the sequence <a href="glossary.html#Alignment" title="A comparison between two or more sequences by matching identical and/or similar residues and assigning a score to the match." class="lexicon-term active">alignment</a> it represents. The dotplot displays detailed comparison of two sequences. Every residue in one sequence is compared to every residue in the other sequence. The first sequence runs along the x-axis and the second sequence along the y-axis. In regions where the two sequences are similar to each other, a row of high scores will run diagonally across the dot matrix. If you're comparing a sequence against itself to find internal repeats, you'll notice that the main diagonal scores maximally, since it's the 100% perfect self-match. To make the score matrix more intelligible, the pairwise scores are averaged over a sliding window that runs diagonally. The averaged score matrix forms a three-dimensional landscape, with the two sequences in two dimensions and the height of the peaks in the third. This landscape is projected onto two dimensions by aid of grayscales - higher peaks are indicated by darker grays. Dotter was written by Erik L.L. Sonnhammer and Richard Durbin Gene 167: GC1-10 (1995)</p>
         </dd>
<dt>
<a id="DUST"></a>
  <a href="glossary/dust.html">DUST</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522DUST%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A standalone application that looks for low complexity sequences.</p>
         </dd>
<dt>
<a id="DWGA_Derived_from_Whole_Genome_Alignments"></a>
  <a href="glossary/dwga-derived-whole-genome-alignments.html">DWGA (Derived from Whole Genome Alignments)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522DWGA (Derived from Whole Genome Alignments)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Human versus Chimpanzee exception: The human versus chimpanzee orthologue predictions were obtained in a completely different manner. Since the current chimpanzee genome sequence <a href="glossary.html#Assembly" title="When the genome or transcriptome of a species is sequenced, typically many short random fragments are sequenced and reassembled by a computer algorithm into longer contiguous sequences called contigs. Genomic contigs may be assembled into longer sequences called scaffolds and sometimes, if the depth of sequencing is high enough, there may be enough information to assemble most of the scaffolds into chromosomes." class="lexicon-term active">assembly</a> is the result of low-coverage sequencing, the assembled sequence is of too poor quality to generate a <a href="glossary.html#Gene_set" title="The set of predicted genes which represent the reference for an assembly. Gene sets contain both protein-coding and ncRNA loci. " class="lexicon-term active">gene set</a> on the classical Ensembl gene build pipeline. The chimpanzee gene set produced by Ensembl has rather been generated by "projecting" human genes to the chimpanzee genome through whole genome BLASTz alignments between both species and filtering for orthologue sequence alignments. The result of this procedure is de facto the human - chimpanzee orthologue set that has been Derived from Whole Genome Alignments (DWGA). See the Prediction Method section on a relevant Ensembl Gene Report page.</p>
         </dd>
  </dl>
                  <a id="letter_e"></a>
  <h2 class="lexicon-letter">E</h2>
  <dl>
    <dt>
<a id="EMBL_European_Molecular_Biology_Laboratory"></a>
  <a href="glossary/embl-european-molecular-biology-laboratory.html">EMBL (European Molecular Biology Laboratory)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522EMBL (European Molecular Biology Laboratory)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Europe's primary nucleotide sequence resource. The main sources of the DNA and RNA sequences in the database are submissions from individual researchers, genome sequencing projects and patent applications.</p>
         </dd>
<dt>
<a id="End_phase"></a>
  <a href="glossary/end-phase.html">End phase</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522End phase%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>In protein-coding exons, it is usually the case that <a href="glossary.html#End_phase" title="In protein-coding exons, it is usually the case that end phase = (phase + exon_length)%3 but end_phase could be -1 if the exon is half-coding and its 3 prime end is UTR." class="lexicon-term active">end phase</a> = (phase + <a href="glossary.html#Exon" title="The part of the genomic sequence that remains in the transcript (mRNA) after introns have been spliced out." class="lexicon-term active">exon</a>_length)%3 but end_phase could be -1 if the exon is half-coding and its 3 prime end is UTR.</p>
         </dd>
<dt>
<a id="Endophagic"></a>
  <a href="glossary/endophagic.html">Endophagic</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Endophagic%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Behavioural trait for feeding indoors.</p>
         </dd>
<dt>
<a id="Endophilic"></a>
  <a href="glossary/endophilic.html">Endophilic</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Endophilic%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Tendency to inhabit/rest indoors.</p>
         </dd>
<dt>
<a id="Ensembl_genes"></a>
  <a href="glossary/ensembl-genes.html">Ensembl genes</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Ensembl genes%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Set of Ensembl gene predictions based on experimental evidence from protein sequences and/or near-full-length cDNA available from public sequence databases. "Ensembl known genes" are predicted on the basis of species-specific database entries from manually curated <a href="glossary.html#UniProtSwiss-Prot" title="(Universal Protein Resource) is the world&#039;s most comprehensive catalogue of information on proteins. UniProt/Swiss-Prot is a curated protein sequence database that provides a high level of annotation, a minimal level of redundancy and high level of integration with other databases. SwissProt is maintained collaboratively by the Swiss Institute for Bioinformatics (SIB) and the European Bioinformatics Institute (EBI)." class="lexicon-term active">UniProt/Swiss-Prot</a>, partially manually curated <a href="glossary.html#RefSeq" title="NCBI&#039;s Reference Sequences (RefSeq) database is a curated database of Genbank&#039;s genomes, mRNAs and proteins. RefSeq attempts to provide a comprehensive, integrated, non-redundant set of sequences, including genomic DNA, tRNA, and protein products, providing a stable reference for gene identification and characterization, mutation analysis, expression studies, polymorphism discovery, and comparative analyses." class="lexicon-term active">RefSeq</a> and <a href="glossary.html#UniProtTrEMBL" title="SPTrEMBL is a subset of TrEMBL (Translated EMBL database) containing the computer-annotated protein translations of all coding sequences (CDS) present in the EMBL EMBL nucleotides that are not yet incorporated into the UniProt/SwissProt database." class="lexicon-term active">UniProt/TrEMBL</a> databases. Predictions of "Ensembl novel genes" are based on other experimental evidence such as protein and cDNA sequence information from related species. Golden genes are the result of a merge between a Havana <a href="glossary.html#Transcript" title="Nucleotide sequence resulting from the transcription of the genomic DNA to mRNA. One gene can have different transcripts or splice variants resulting from the alternative splicing of different exons in genes." class="lexicon-term active">transcript</a> (manually curated) and an Ensembl gene prediction from the annotation pipeline. See "havana transcript".</p>
         </dd>
<dt>
<a id="Eponine"></a>
  <a href="glossary/eponine.html">Eponine</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Eponine%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p><a href="glossary.html#Eponine" title="Eponine is a probabilistic method for detecting transcription start sites (TSS) in mammalian genomic sequence, with good specificity and excellent positional accuracy. Eponine models consist of a set of DNA weight matrices recognizing specific sequence motifs. Each of these is associated with a position distribution relative to the TSS." class="lexicon-term active">Eponine</a> is a probabilistic method for detecting transcription start sites (TSS) in mammalian genomic sequence, with good specificity and excellent positional accuracy. Eponine models consist of a set of DNA weight matrices recognizing specific sequence motifs. Each of these is associated with a position distribution relative to the TSS.</p>
         </dd>
<dt>
<a id="EST_Expressed_Sequence_Tags"></a>
  <a href="glossary/est-expressed-sequence-tags.html">EST (Expressed Sequence Tags)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522EST (Expressed Sequence Tags)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Coarse sequence reads from flanking vector regions into the inserts of cDNA libraries. ESTs act as physical markers for cloning and full length sequencing of the cDNAs of expressed genes. Typically identified by purifying mRNAs, converting to cDNAs, and then sequencing a portion of the cDNAs. Usually short, single reads from a tissue or stage in development.</p>
         </dd>
<dt>
<a id="EST_genes"></a>
  <a href="glossary/est-genes.html">EST genes</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522EST genes%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Set of Ensembl gene predictions solely based on EST evidence. The process of EST gene prediction uses a combination of <a href="glossary.html#Exonerate" title="A fast gapped DNA-DNA alignment algorithm. It can be used for aligning various types of sequences such as genomic DNA, cDNAs/ESTs, and proteins." class="lexicon-term active">Exonerate</a>, <a href="glossary.html#BLAST" title="BLAST (Basic Local Alignment Search Tool) is a sequence comparison algorithm optimised for speed which is used to search sequence databases for optimal local alignments to a query. (Altschul et al., J Mol Biol 215:403-410; 1990)" class="lexicon-term active">BLAST</a> and Est2Genome to map ESTs onto the genomic sequence. Redundant ESTs are merged, before GenomeWise is used to assign 5' and 3' UTRs to the longest found ORF. See Eyras et al. for a more complete explanation of the EST gene prediction process.</p>
         </dd>
<dt>
<a id="Exon"></a>
  <a href="glossary/exon.html">Exon</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Exon%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>The part of the genomic sequence that remains in the <a href="glossary.html#Transcript" title="Nucleotide sequence resulting from the transcription of the genomic DNA to mRNA. One gene can have different transcripts or splice variants resulting from the alternative splicing of different exons in genes." class="lexicon-term active">transcript</a> (mRNA) after introns have been spliced out.</p>
         </dd>
<dt>
<a id="Exonerate"></a>
  <a href="glossary/exonerate.html">Exonerate</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Exonerate%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A fast gapped DNA-DNA <a href="glossary.html#Alignment" title="A comparison between two or more sequences by matching identical and/or similar residues and assigning a score to the match." class="lexicon-term active">alignment</a> <a href="glossary.html#Algorithm" title="A sequence of computational tasks or actions that carry out a specific function." class="lexicon-term active">algorithm</a>. It can be used for aligning various types of sequences such as genomic DNA, cDNAs/ESTs, and proteins.</p>
         </dd>
<dt>
<a id="Exophagic"></a>
  <a href="glossary/exophagic.html">Exophagic</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Exophagic%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Behavioural trait for feeding oudoors.</p>
         </dd>
<dt>
<a id="Exophilic"></a>
  <a href="glossary/exophilic.html">Exophilic</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Exophilic%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Tendency to inhabit/rest in outdoor areas.</p>
         </dd>
  </dl>
                  <a id="letter_f"></a>
  <h2 class="lexicon-letter">F</h2>
  <dl>
    <dt>
<a id="Feature"></a>
  <a href="glossary/feature.html">Feature</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Feature%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Any annotation on a specific location in the genomic sequence.</p>
         </dd>
<dt>
<a id="Fgenes"></a>
  <a href="glossary/fgenes.html">Fgenes</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Fgenes%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p><a href="glossary.html#Fgenes" title="FGENES, also known as Find Genes, is a Human gene predictor that is based on pattern recognition of different types of exons, promoters and poly A signals. It is built based on linear discriminant functions of internal, 5&#039;-coding, and 3&#039;-coding exon recognition. It is designed to find the optimal combination of these components and to construct a set of gene models along a given sequence." class="lexicon-term active">FGENES</a>, also known as Find Genes, is a Human gene predictor that is based on pattern recognition of different types of exons, promoters and poly A signals. It is built based on linear discriminant functions of internal, 5'-coding, and 3'-coding <a href="glossary.html#Exon" title="The part of the genomic sequence that remains in the transcript (mRNA) after introns have been spliced out." class="lexicon-term active">exon</a> recognition. It is designed to find the optimal combination of these components and to construct a set of gene models along a given sequence.</p>
         </dd>
<dt>
<a id="Flanking_sequence"></a>
  <a href="glossary/flanking-sequence.html">Flanking sequence</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Flanking sequence%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Sequence 5' or 3' to a DNA or RNA sequence of interest (for example gene, <a href="glossary.html#Transcript" title="Nucleotide sequence resulting from the transcription of the genomic DNA to mRNA. One gene can have different transcripts or splice variants resulting from the alternative splicing of different exons in genes." class="lexicon-term active">transcript</a>, SNP or <a href="glossary.html#repeat" title="Repetitive DNA in which the same sequence occurs multiple times." class="lexicon-term active">repeat</a>).</p>
         </dd>
<dt>
<a id="Frameshift_intron"></a>
  <a href="glossary/frameshift-intron.html">Frameshift intron</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Frameshift intron%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Frameshift introns are the length of 1, 2, 4, or 5 basepairs. They are introduced by the Ensembl <a href="glossary.html#Genebuild" title="The Ensembl gene annotation system is used to annotate genome assemblies. This process is called the genebuild." class="lexicon-term active">genebuild</a> in order to fit the cDNA sequence to the genome.</p>
         </dd>
  </dl>
                  <a id="letter_g"></a>
  <h2 class="lexicon-letter">G</h2>
  <dl>
    <dt>
<a id="GenBank_WGS"></a>
  <a href="glossary/genbank-wgs.html">GenBank WGS</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522GenBank WGS%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>GenBank accession for a Whole Genome Shotgun (WGS) project. This is the master accession which should link to both contigs and <a href="glossary.html#Scaffold" title="Supercontigs or scaffolds are sets of ordered, oriented contigs. They are longer sequences than contigs, but shorter than full chromosomes." class="lexicon-term active">scaffold</a> data for the species. If an <a href="glossary.html#Assembly" title="When the genome or transcriptome of a species is sequenced, typically many short random fragments are sequenced and reassembled by a computer algorithm into longer contiguous sequences called contigs. Genomic contigs may be assembled into longer sequences called scaffolds and sometimes, if the depth of sequencing is high enough, there may be enough information to assemble most of the scaffolds into chromosomes." class="lexicon-term active">assembly</a> is changed the WGS accession will be modified. It is possible that the assembly improvement involves re-scaffolding and the underlying <a href="glossary.html#Contig" title="A contig is a contiguous stretch of DNA sequence without gaps that has been assembled solely based on direct sequencing information.    Short sequences (reads) from a fragmented genome are compared against one another, and overlapping reads are merged to produce one long sequence. This merging process is iterative: overlapping reads are added to the merged sequence whenever possible and so the merged sequence becomes even longer. When no further reads overlap the long merged sequence, then this sequence - called a contig - has reached its maximum length.    Contig can be used in other contexts: A contig can be the sequence corresponding to only one clone. A contig map shows the regions of a chromosome where contiguous DNA segments overlap." class="lexicon-term active">contig</a> sequences can be unchanged.</p>
         </dd>
<dt>
<a id="Gene_set"></a>
  <a href="glossary/gene-set.html">Gene set</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Gene set%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>The set of predicted genes which represent the reference for an <a href="glossary.html#Assembly" title="When the genome or transcriptome of a species is sequenced, typically many short random fragments are sequenced and reassembled by a computer algorithm into longer contiguous sequences called contigs. Genomic contigs may be assembled into longer sequences called scaffolds and sometimes, if the depth of sequencing is high enough, there may be enough information to assemble most of the scaffolds into chromosomes." class="lexicon-term active">assembly</a>. Gene sets contain both protein-coding and ncRNA loci.</p>
         </dd>
<dt>
<a id="Genebuild"></a>
  <a href="glossary/genebuild.html">Genebuild</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Genebuild%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>The Ensembl gene annotation system is used to annotate genome assemblies. This process is called the <a href="glossary.html#Genebuild" title="The Ensembl gene annotation system is used to annotate genome assemblies. This process is called the genebuild." class="lexicon-term active">genebuild</a>.</p>
         </dd>
<dt>
<a id="GeneWise"></a>
  <a href="glossary/genewise.html">GeneWise</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522GeneWise%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p><a href="glossary.html#GeneWise" title="GeneWise is sequence analysis tool for comparing proteins or profile HMMs to DNA sequences allowing for introns and frameshifts. The Wise2 package was written by Ewan Birney. More information about the package can be obtained at: http://www.ebi.ac.uk/Tools/psa/genewise/help/" class="lexicon-term active">GeneWise</a> is sequence analysis tool for comparing proteins or profile HMMs to DNA sequences allowing for introns and frameshifts. The Wise2 package was written by Ewan Birney. More information about the package can be obtained at: <a href="http://www.ebi.ac.uk/Tools/psa/genewise/help/">http://www.ebi.ac.uk/Tools/psa/genewise/help/</a></p>
         </dd>
<dt>
<a id="Genomic_marker"></a>
  <a href="glossary/genomic-marker.html">Genomic marker</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Genomic marker%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A short sequence whose placement on the genome is known.</p>
         </dd>
<dt>
<a id="Genotype"></a>
  <a href="glossary/genotype.html">Genotype</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Genotype%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Specific alleles present in an individual's genome, or the genetic makeup of one organism.</p>
         </dd>
<dt>
<a id="GENSCAN"></a>
  <a href="glossary/genscan.html">GENSCAN</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522GENSCAN%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>An application for identification of complete gene structures in genomic DNA (Burge, C. and Karlin, S. (1997) Prediction of complete gene structures in human genomic DNA. J. Mol. Biol. 268, 78-94). The splice site models used are described in more detail in: Burge, C. B. (1998) Modeling dependencies in pre-mRNA splicing signals. In Salzberg, S., Searls, D. and Kasif, S., eds. Computational Methods in Molecular Biology, Elsevier Science, Amsterdam, pp. 127-163.</p>
         </dd>
<dt>
<a id="GO_Gene_Ontology"></a>
  <a href="glossary/go-gene-ontology.html">GO (Gene Ontology)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522GO (Gene Ontology)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>An organized hierarchy of terms produced by the <a href="http://www.geneontology.org/">Gene Ontology Consortium</a>, used to describe biological processes, cellular component, and molecular function. Specific GO terms are as follows: Molecular Function Ontology. Tasks performed by individual gene products; examples are carbohydrate binding and ATPase activity. Biological Process Ontology. Broad biological goals, such as mitosis or purine metabolism, are accomplished by ordered assemblies of molecular functions. Cellular Component Ontology. Subcellular structures, locations, and macromolecular complexes; examples include nucleus, telomere, and origin recognition complex. A gene may be indexed under many GO terms depending on GO classification system. A gene product has one or more molecular functions and is used in one or more biological processes; it might be associated with one or more cellular components. For instance, cytochrome c can be described by the molecular function term electron transporter activity, the biological process terms oxidative phosphorylation and induction of cell death, and the cellular component terms mitochondrial matrix and mitochondrial inner membrane.</p>
         </dd>
<dt>
<a id="Golden_path_length"></a>
  <a href="glossary/golden-path-length.html">Golden path length</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Golden path length%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>The golden path is the length of the reference <a href="glossary.html#Assembly" title="When the genome or transcriptome of a species is sequenced, typically many short random fragments are sequenced and reassembled by a computer algorithm into longer contiguous sequences called contigs. Genomic contigs may be assembled into longer sequences called scaffolds and sometimes, if the depth of sequencing is high enough, there may be enough information to assemble most of the scaffolds into chromosomes." class="lexicon-term active">assembly</a>. It consists of the sum of all top-level sequences in the seq_region table, omitting any redundant regions such as <a href="glossary.html#Haplotypes" title="A set of genes or markers on one chromosome that are inherited together. Often refers to SNPs that are closely linked (i.e. have a high linkage disequilibrium (LD) value, and are inherited together.) In Region in Detail, the haplotype regions are coloured with a red background." class="lexicon-term active">haplotypes</a> and PARs (pseudoautosomal regions).</p>
         </dd>
  </dl>
                  <a id="letter_h"></a>
  <h2 class="lexicon-letter">H</h2>
  <dl>
    <dt>
<a id="Haplotype"></a>
  <a href="glossary/haplotype.html">Haplotype</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Haplotype%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Known variations to the primary <a href="glossary.html#Assembly" title="When the genome or transcriptome of a species is sequenced, typically many short random fragments are sequenced and reassembled by a computer algorithm into longer contiguous sequences called contigs. Genomic contigs may be assembled into longer sequences called scaffolds and sometimes, if the depth of sequencing is high enough, there may be enough information to assemble most of the scaffolds into chromosomes." class="lexicon-term active">assembly</a>, due to variability in the human genome sequence (eg. the highly variable MHC locus containing halpotypes HSCHR6_MHC_COX, HSCHR6_MHC_SSTO, HSCHR6_MHC_APD, HSCHR6_MHC_DBB, HSCHR6_MHC_MANN, HSCHR6_MHC_MCF, and HSCHR6_MHC_QBL). In Region in Detail, the <a href="glossary.html#Haplotype" title="Known variations to the primary assembly, due to variability in the human genome sequence (eg. the highly variable MHC locus containing halpotypes HSCHR6_MHC_COX, HSCHR6_MHC_SSTO, HSCHR6_MHC_APD, HSCHR6_MHC_DBB, HSCHR6_MHC_MANN, HSCHR6_MHC_MCF, and HSCHR6_MHC_QBL). In Region in Detail, the haplotype regions are coloured with a red background." class="lexicon-term active">haplotype</a> regions are coloured with a red background.</p>
         </dd>
<dt>
<a id="Haplotypes"></a>
  <a href="glossary/haplotypes.html">Haplotypes</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Haplotypes%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A set of genes or markers on one chromosome that are inherited together. Often refers to SNPs that are closely linked (i.e. have a high <a href="glossary.html#Linkage" title="A measure of how often features (genes, specific sequences) on a chromosome are inherited together." class="lexicon-term active">linkage</a> disequilibrium (LD) value, and are inherited together.) In Region in Detail, the <a href="glossary.html#Haplotype" title="Known variations to the primary assembly, due to variability in the human genome sequence (eg. the highly variable MHC locus containing halpotypes HSCHR6_MHC_COX, HSCHR6_MHC_SSTO, HSCHR6_MHC_APD, HSCHR6_MHC_DBB, HSCHR6_MHC_MANN, HSCHR6_MHC_MCF, and HSCHR6_MHC_QBL). In Region in Detail, the haplotype regions are coloured with a red background." class="lexicon-term active">haplotype</a> regions are coloured with a red background.</p>
         </dd>
<dt>
<a id="High-coverage_genome"></a>
  <a href="glossary/high-coverage-genome.html">High-coverage genome</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522High-coverage genome%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Refers to the number of overlapping sequences used to build the genomic <a href="glossary.html#Assembly" title="When the genome or transcriptome of a species is sequenced, typically many short random fragments are sequenced and reassembled by a computer algorithm into longer contiguous sequences called contigs. Genomic contigs may be assembled into longer sequences called scaffolds and sometimes, if the depth of sequencing is high enough, there may be enough information to assemble most of the scaffolds into chromosomes." class="lexicon-term active">assembly</a>. High coverage, such as human and mouse genomes, indicates a good amount of sequence information. This is also referred to as deep-coverage. Low coverage reflects a low amount of sequence information.</p>
         </dd>
<dt>
<a id="homologues"></a>
  <a href="glossary/homologues.html">homologues</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522homologues%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Specific sequences that are descended from the same common sequence in an ancestor. See orthologues or paralogues.</p>
         </dd>
  </dl>
                  <a id="letter_i"></a>
  <h2 class="lexicon-letter">I</h2>
  <dl>
    <dt>
<a id="In-del_Insertion-deletion"></a>
  <a href="glossary/del-insertion-deletion.html">In-del (Insertion-deletion)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522In-del (Insertion-deletion)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A <a href="glossary.html#Mutation" title="A modification (insertion, deletion, or alteration) in the genomic or amino acid sequence." class="lexicon-term active">mutation</a> or polymorphism in which one or more base pairs have been inserted into or removed from a genomic sequence.</p>
         </dd>
<dt>
<a id="InterPro"></a>
  <a href="glossary/interpro.html">InterPro</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522InterPro%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p><a href="glossary.html#InterPro" title="InterPro is an integrated resource for protein families, domains and sites, combining information from several different protein signature databases. InterPro IDs are linked to the summary of information about that domain or family. InterPro is managed by EBI. A number of databases (SwissProt, TrEMBL, PROSITE, PRINTS, Pfam, and ProDom, SMART, TIGRFAMs, PIR SuperFamilies and SUPERFAMILY) with different approaches to biological information are used to derive protein signatures. ProteinView, GeneView and DomainView provide links to the relevant InterPro entries." class="lexicon-term active">InterPro</a> is an integrated resource for protein families, domains and sites, combining information from several different protein signature databases. InterPro IDs are linked to the summary of information about that <a href="glossary.html#Domain" title="A region of special biological interest within a single protein sequence. However, a domain may also be defined as a region within the three-dimensional structure of a protein that may encompass regions of several distinct protein sequences that accomplishes a specific function. A domain class is a group of domains that share a common set of well-defined properties or characteristics." class="lexicon-term active">domain</a> or family. InterPro is managed by EBI. A number of databases (SwissProt, TrEMBL, <a href="glossary.html#Prosite" title="PROSITE is a database of protein families and domains run by the (Expert Protein Analysis System (ExPASy) proteomics server of the Swiss Institute of Bioinformatics (SIB). It consists of biologically significant sites, patterns and profiles that help to reliably identify to which known protein family (if any) a new sequence belongs." class="lexicon-term active">PROSITE</a>, <a href="glossary.html#Prints" title="The PRINTS protein fingerprint database is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of a SwissProt/TrEMBL composite. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs, full diagnostic potency deriving from the mutual context provided by motif neighbors." class="lexicon-term active">PRINTS</a>, <a href="glossary.html#Pfam" title="Pfam is a large collection of multiple sequence alignments and hidden Markov models covering many common protein domains and families. Pfam can be used to view the domain organization of proteins, to view multiple alignments, protein domain architectures, protein structures, and species distributions." class="lexicon-term active">Pfam</a>, and ProDom, SMART, TIGRFAMs, PIR SuperFamilies and SUPERFAMILY) with different approaches to biological information are used to derive protein signatures. ProteinView, GeneView and DomainView provide links to the relevant InterPro entries.</p>
         </dd>
<dt>
<a id="Intron"></a>
  <a href="glossary/intron.html">Intron</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Intron%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>The part of the genomic sequence that is transcribed and then spliced out of the <a href="glossary.html#Transcript" title="Nucleotide sequence resulting from the transcription of the genomic DNA to mRNA. One gene can have different transcripts or splice variants resulting from the alternative splicing of different exons in genes." class="lexicon-term active">transcript</a> (mRNA). Noncoding.</p>
         </dd>
  </dl>
                  <a id="letter_j"></a>
  <h2 class="lexicon-letter">J</h2>
  <dl>
    <dt>
<a id="Jalview"></a>
  <a href="glossary/jalview.html">Jalview</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Jalview%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p><a href="glossary.html#Jalview" title="Jalview is a multiple alignment editor, used by the EBI clustalw server and the PFAM protein domain database and is available as a general purpose alignment editor." class="lexicon-term active">Jalview</a> is a multiple <a href="glossary.html#Alignment" title="A comparison between two or more sequences by matching identical and/or similar residues and assigning a score to the match." class="lexicon-term active">alignment</a> editor, used by the EBI clustalw server and the <a href="glossary.html#Pfam" title="Pfam is a large collection of multiple sequence alignments and hidden Markov models covering many common protein domains and families. Pfam can be used to view the domain organization of proteins, to view multiple alignments, protein domain architectures, protein structures, and species distributions." class="lexicon-term active">PFAM</a> protein <a href="glossary.html#Domain" title="A region of special biological interest within a single protein sequence. However, a domain may also be defined as a region within the three-dimensional structure of a protein that may encompass regions of several distinct protein sequences that accomplishes a specific function. A domain class is a group of domains that share a common set of well-defined properties or characteristics." class="lexicon-term active">domain</a> database and is available as a general purpose alignment editor.</p>
         </dd>
  </dl>
                  <a id="letter_k"></a>
  <h2 class="lexicon-letter">K</h2>
  <dl>
    <dt>
<a id="Karyotype"></a>
  <a href="glossary/karyotype.html">Karyotype</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Karyotype%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>The <a href="glossary.html#Karyotype" title="The Karyotype View in VectorBase displays the set of chromosomes for a species, including the centromere and banding pattern as they would appear under a light microscope. Dark bands indicate heterochromatin and light bands indicate homochromatin.
The Karyotype View is only available for species where the genome assembly provided to us has been assembled into chromosomes, currently only Anopheles gambiae. For many species in VectorBase, the genome assembly is comprised of only unplaced scaffolds." class="lexicon-term active">Karyotype</a> View in VectorBase displays the set of chromosomes for a species, including the centromere and banding pattern as they would appear under a light microscope. Dark bands indicate heterochromatin and light bands indicate homochromatin.<br />
The Karyotype View is only available for species where the genome <a href="glossary.html#Assembly" title="When the genome or transcriptome of a species is sequenced, typically many short random fragments are sequenced and reassembled by a computer algorithm into longer contiguous sequences called contigs. Genomic contigs may be assembled into longer sequences called scaffolds and sometimes, if the depth of sequencing is high enough, there may be enough information to assemble most of the scaffolds into chromosomes." class="lexicon-term active">assembly</a> provided to us has been assembled into chromosomes, currently only <em>Anopheles gambiae</em>. For many species in VectorBase, the genome assembly is comprised of only unplaced scaffolds.</p>
         </dd>
<dt>
<a id="Known_gene"></a>
  <a href="glossary/known-gene.html">Known gene</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Known gene%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A <a href="glossary.html#Known_gene" title="A known gene is an Ensembl gene for which at least one known transcript has been annotated." class="lexicon-term active">known gene</a> is an Ensembl gene for which at least one <a href="glossary.html#Known_transcript" title="A known Ensembl transcript matches to a sequence for the same species in a public, scientific database such as UniProtKB or NCBI RefSeq." class="lexicon-term active">known transcript</a> has been annotated.</p>
         </dd>
<dt>
<a id="Known_transcript"></a>
  <a href="glossary/known-transcript.html">Known transcript</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Known transcript%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A known Ensembl <a href="glossary.html#Transcript" title="Nucleotide sequence resulting from the transcription of the genomic DNA to mRNA. One gene can have different transcripts or splice variants resulting from the alternative splicing of different exons in genes." class="lexicon-term active">transcript</a> matches to a sequence for the same species in a public, scientific database such as UniProtKB or NCBI <a href="glossary.html#RefSeq" title="NCBI&#039;s Reference Sequences (RefSeq) database is a curated database of Genbank&#039;s genomes, mRNAs and proteins. RefSeq attempts to provide a comprehensive, integrated, non-redundant set of sequences, including genomic DNA, tRNA, and protein products, providing a stable reference for gene identification and characterization, mutation analysis, expression studies, polymorphism discovery, and comparative analyses." class="lexicon-term active">RefSeq</a>.</p>
         </dd>
  </dl>
                  <a id="letter_l"></a>
  <h2 class="lexicon-letter">L</h2>
  <dl>
    <dt>
<a id="LD_Linkage_Disequilibrium"></a>
  <a href="glossary/ld-linkage-disequilibrium.html">LD (Linkage Disequilibrium)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522LD (Linkage Disequilibrium)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A measure of how often two SNPs or specific sequences are inherited together.</p>
         </dd>
<dt>
<a id="Length_aa"></a>
  <a href="glossary/length-aa.html">Length (aa)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Length (aa)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>The number of amino acids in, for example, a protein.</p>
         </dd>
<dt>
<a id="Length_bp"></a>
  <a href="glossary/length-bp.html">Length (bp)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Length (bp)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>The number of base pairs in, for example, a <a href="glossary.html#Transcript" title="Nucleotide sequence resulting from the transcription of the genomic DNA to mRNA. One gene can have different transcripts or splice variants resulting from the alternative splicing of different exons in genes." class="lexicon-term active">transcript</a>.</p>
         </dd>
<dt>
<a id="LincRNA"></a>
  <a href="glossary/lincrna.html">LincRNA</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522LincRNA%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Large intergenic non-coding RNAs, usually associated with open chromatin signatures such as histone modification sites.</p>
         </dd>
<dt>
<a id="Linkage"></a>
  <a href="glossary/linkage.html">Linkage</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Linkage%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A measure of how often features (genes, specific sequences) on a chromosome are inherited together.</p>
         </dd>
<dt>
<a id="lncRNA"></a>
  <a href="glossary/lncrna.html">lncRNA</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522lncRNA%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
        A gene that encodes a long non-coding RNA. <a href="http://www.sequenceontology.org/browser/current_svn/term/SO:0002127">Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0002127</a>        </dd>
<dt>
<a id="Low-complexity_region"></a>
  <a href="glossary/low-complexity-region.html">Low-complexity region</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Low-complexity region%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A region in the sequence with a biased composition (i.e. repeated sequences or residues.)</p>
         </dd>
<dt>
<a id="Low-coverage_genome"></a>
  <a href="glossary/low-coverage-genome.html">Low-coverage genome</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Low-coverage genome%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Refers to the number of overlapping sequences used to build the genomic <a href="glossary.html#Assembly" title="When the genome or transcriptome of a species is sequenced, typically many short random fragments are sequenced and reassembled by a computer algorithm into longer contiguous sequences called contigs. Genomic contigs may be assembled into longer sequences called scaffolds and sometimes, if the depth of sequencing is high enough, there may be enough information to assemble most of the scaffolds into chromosomes." class="lexicon-term active">assembly</a>. High coverage, such as human and mouse genomes, indicates a good amount of sequence information. This is also referred to as deep-coverage. Low coverage, such as the lesser-known mammals, reflects a low amount of sequence information. 2X genomes are low coverage.</p>
         </dd>
  </dl>
                  <a id="letter_m"></a>
  <h2 class="lexicon-letter">M</h2>
  <dl>
    <dt>
<a id="MBRH_Multiple_Best_Reciprocal_Hit"></a>
  <a href="glossary/mbrh-multiple-best-reciprocal-hit.html">MBRH (Multiple Best Reciprocal Hit)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522MBRH (Multiple Best Reciprocal Hit)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>When due to gene duplications there are multiple 'best' hits with identical score, E-value, % identity, %positivity, one is unable to pick a unique orthologue for a gene. This results in more complex graphs of 'best' relationships. This often occurs when different genes have identical translations, which could be due to a duplication event, an <a href="glossary.html#Assembly" title="When the genome or transcriptome of a species is sequenced, typically many short random fragments are sequenced and reassembled by a computer algorithm into longer contiguous sequences called contigs. Genomic contigs may be assembled into longer sequences called scaffolds and sometimes, if the depth of sequencing is high enough, there may be enough information to assemble most of the scaffolds into chromosomes." class="lexicon-term active">assembly</a> error, or chance. On average 3% of the genes have an identical translation to some other gene either within it's genome or in another genome.        * MBRH / DUP 1.# - MBRH set where in one genome there is only one gene, but the other genome has multiple genes, all on the same chromosome and within 1.5 megbases of each other. This could be due to recent gene duplication events where sequences have not diverged or a mis-assembly of the genome sequence leading to artificial, apparent gene duplications. (e.g. MBRH / DUP 1.2 or MBRH/ DUP 1.4)        * MBRH / SYN - This is a more complex MBRH set where there are multiple genes in each genome split across multiple chromosomes. The one(s) labeled MBRH/SYN satisfies both the MBRH criteria and the RHS search criteria.        * MBRH / COMPLEX - This is a more complex MBRH set where there are multiple genes in each genome split across multiple chromosomes. This MBRH pair does not satisfy the RHS criteria.</p>
         </dd>
<dt>
<a id="Microsatellite"></a>
  <a href="glossary/microsatellite.html">Microsatellite</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Microsatellite%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A region in the genomic sequence containing short <a href="glossary.html#tandem_repeats" title="Multiple copies of the same base sequence on a chromosome; used as markers in physical mapping." class="lexicon-term active">tandem repeats</a>.</p>
         </dd>
<dt>
<a id="miRNA"></a>
  <a href="glossary/mirna.html">miRNA</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522miRNA%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>MicroRNA is single-stranded RNA, typically 21-23 base pairs long, that is thought to be involved in gene regulation (especially inhibition of protein expression).</p>
         </dd>
<dt>
<a id="miRNA_pseudogene"></a>
  <a href="glossary/mirna-pseudogene.html">miRNA pseudogene</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522miRNA pseudogene%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>MicroRNA <a href="glossary.html#Pseudogene" title="A noncoding sequence similar to an active protein. A sequence that closely resembles a known functional gene, at another locus within a genome, that is non-functional as a consequence of (usually several) mutations that prevent either its transcription or translation (or both). In general, pseudogenes result from either reverse transcription of a transcript of their &quot;normal&quot; paralog (SO:0000043) (in which case the pseudogene typically lacks introns and includes a poly(A) tail) or from recombination (SO:0000044) (in which case the pseudogene is typically a tandem duplication of its &quot;normal&quot; paralog). Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000336" class="lexicon-term active">pseudogene</a>.</p>
         </dd>
<dt>
<a id="Misc_RNA"></a>
  <a href="glossary/misc-rna.html">Misc RNA</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Misc RNA%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Miscellaneous RNA.</p>
         </dd>
<dt>
<a id="Misc_RNA_pseudogene"></a>
  <a href="glossary/misc-rna-pseudogene.html">Misc RNA pseudogene</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Misc RNA pseudogene%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Miscellaneous RNA <a href="glossary.html#Pseudogene" title="A noncoding sequence similar to an active protein. A sequence that closely resembles a known functional gene, at another locus within a genome, that is non-functional as a consequence of (usually several) mutations that prevent either its transcription or translation (or both). In general, pseudogenes result from either reverse transcription of a transcript of their &quot;normal&quot; paralog (SO:0000043) (in which case the pseudogene typically lacks introns and includes a poly(A) tail) or from recombination (SO:0000044) (in which case the pseudogene is typically a tandem duplication of its &quot;normal&quot; paralog). Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000336" class="lexicon-term active">pseudogene</a>.</p>
         </dd>
<dt>
<a id="Motif"></a>
  <a href="glossary/motif.html">Motif</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Motif%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A conserved region of sequence with a specific function/ structure.</p>
         </dd>
<dt>
<a id="mt_rRNA"></a>
  <a href="glossary/mt-rrna.html">mt rRNA</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522mt rRNA%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Mitochondrial ribosomal RNA.</p>
         </dd>
<dt>
<a id="mt_tRNA"></a>
  <a href="glossary/mt-trna.html">mt tRNA</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522mt tRNA%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Mitochondrial transfer RNA.</p>
         </dd>
<dt>
<a id="Mt_tRNA_pseudogene"></a>
  <a href="glossary/mt-trna-pseudogene.html">Mt tRNA pseudogene</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Mt tRNA pseudogene%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Mitochondrial transfer RNA <a href="glossary.html#Pseudogene" title="A noncoding sequence similar to an active protein. A sequence that closely resembles a known functional gene, at another locus within a genome, that is non-functional as a consequence of (usually several) mutations that prevent either its transcription or translation (or both). In general, pseudogenes result from either reverse transcription of a transcript of their &quot;normal&quot; paralog (SO:0000043) (in which case the pseudogene typically lacks introns and includes a poly(A) tail) or from recombination (SO:0000044) (in which case the pseudogene is typically a tandem duplication of its &quot;normal&quot; paralog). Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000336" class="lexicon-term active">pseudogene</a>.</p>
         </dd>
<dt>
<a id="Mutation"></a>
  <a href="glossary/mutation.html">Mutation</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Mutation%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A modification (insertion, deletion, or alteration) in the genomic or amino acid sequence.</p>
         </dd>
  </dl>
                  <a id="letter_n"></a>
  <h2 class="lexicon-letter">N</h2>
  <dl>
    <dt>
<a id="ncRNA_non-coding_RNA"></a>
  <a href="glossary/ncrna-non-coding-rna.html">ncRNA (non-coding RNA)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522ncRNA (non-coding RNA)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Short non-coding RNAs such as <a href="glossary.html#rRNA" title="Ribosomal RNA. RNA that comprises part of a ribosome, and that can provide both structural scaffolding and catalytic activity. Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000252" class="lexicon-term active">rRNA</a>, <a href="glossary.html#ScRNA" title="Small cytoplasmic RNA." class="lexicon-term active">scRNA</a>, snTNA, <a href="glossary.html#snoRNA" title="Small nucleolar RNA, involved in modifications of other RNAs. A snoRNA (small nucleolar RNA) is any one of a class of small RNAs that are associated with the eukaryotic nucleus as components of small nucleolar ribonucleoproteins. They participate in the processing or modifications of many RNAs, mostly ribosomal RNAs (rRNAs) though snoRNAs are also known to target other classes of RNA, including spliceosomal RNAs, tRNAs, and mRNAs via a stretch of sequence that is complementary to a sequence in the targeted RNA. Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000275" class="lexicon-term active">snoRNA</a> and <a href="glossary.html#miRNA" title="MicroRNA is single-stranded RNA, typically 21-23 base pairs long, that is thought to be involved in gene regulation (especially inhibition of protein expression)." class="lexicon-term active">miRNA</a> are annotated by the Ensembl ncRNA pipeline (see article). To view these short ncRNAs, go to Region In Detail and open the Configure This Page window. Select ncRNA from the Genes menu.    Transfer RNAs (tRNAs) are identified by tRNAscan. To view tRNAs, go to Region In Detail and open the Configure This Page window. Select <a href="glossary.html#tRNA" title="Transfer RNA. These are identified using tRNAscan. Transfer RNA (tRNA) molecules are approximately 80 nucleotides in length. Their secondary structure includes four short double-helical elements and three loops (D, anti-codon, and T loops). Further hydrogen bonds mediate the characteristic L-shaped molecular structure. Transfer RNAs have two regions of fundamental functional importance: the anti-codon, which is responsible for specific mRNA codon recognition, and the 3&#039; end, to which the tRNA&#039;s corresponding amino acid is attached (by aminoacyl-tRNA synthetases). Transfer RNAs cope with the degeneracy of the genetic code in two manners: having more than one tRNA (with a specific anti-codon) for a particular amino acid; and &#039;wobble&#039; base-pairing, i.e. permitting non-standard base-pairing at the 3rd anti-codon position. Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000253" class="lexicon-term active">tRNA</a> from the Simple <a href="glossary.html#Feature" title="Any annotation on a specific location in the genomic sequence." class="lexicon-term active">Feature</a> menu.    Long intergenic ncRNAs have only been annotated for human and mouse. To view long ncRNAs, go to Region In Detail and open the Configure This Page window. Select <a href="glossary.html#LincRNA" title="Large intergenic non-coding RNAs, usually associated with open chromatin signatures such as histone modification sites." class="lexicon-term active">lincRNA</a> from the Genes menu. An RNA <a href="glossary.html#Transcript" title="Nucleotide sequence resulting from the transcription of the genomic DNA to mRNA. One gene can have different transcripts or splice variants resulting from the alternative splicing of different exons in genes." class="lexicon-term active">transcript</a> that does not encode for a protein rather the RNA molecule is the gene product. <a href="http://www.sequenceontology.org/browser/current_svn/term/SO:0000655">Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000655</a></p>
         </dd>
<dt>
<a id="Nonsense_mediated_decay"></a>
  <a href="glossary/nonsense-mediated-decay.html">Nonsense mediated decay</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Nonsense mediated decay%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p><a href="glossary.html#Transcript" title="Nucleotide sequence resulting from the transcription of the genomic DNA to mRNA. One gene can have different transcripts or splice variants resulting from the alternative splicing of different exons in genes." class="lexicon-term active">Transcript</a> is thought to undergo <a href="glossary.html#Nonsense_mediated_decay" title="Transcript is thought to undergo nonsense mediated decay, a process which detects nonsense mutations and prevents the expression of truncated or erroneous proteins." class="lexicon-term active">nonsense mediated decay</a>, a process which detects nonsense mutations and prevents the expression of truncated or erroneous proteins.</p>
         </dd>
<dt>
<a id="Novel_gene"></a>
  <a href="glossary/novel-gene.html">Novel gene</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Novel gene%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A <a href="glossary.html#Novel_gene" title="A novel gene is an Ensembl gene for which only one or more novel transcripts have been annotated." class="lexicon-term active">novel gene</a> is an Ensembl gene for which only one or more novel transcripts have been annotated.</p>
         </dd>
<dt>
<a id="Novel_transcript"></a>
  <a href="glossary/novel-transcript.html">Novel transcript</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Novel transcript%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A novel Ensembl <a href="glossary.html#Transcript" title="Nucleotide sequence resulting from the transcription of the genomic DNA to mRNA. One gene can have different transcripts or splice variants resulting from the alternative splicing of different exons in genes." class="lexicon-term active">transcript</a> does not match to a sequence for the same species in a public, scientific database such as UniProtKB or NCBI <a href="glossary.html#RefSeq" title="NCBI&#039;s Reference Sequences (RefSeq) database is a curated database of Genbank&#039;s genomes, mRNAs and proteins. RefSeq attempts to provide a comprehensive, integrated, non-redundant set of sequences, including genomic DNA, tRNA, and protein products, providing a stable reference for gene identification and characterization, mutation analysis, expression studies, polymorphism discovery, and comparative analyses." class="lexicon-term active">RefSeq</a>.</p>
         </dd>
  </dl>
                  <a id="letter_o"></a>
  <h2 class="lexicon-letter">O</h2>
  <dl>
    <dt>
<a id="ORF_Open_Reading_Frame"></a>
  <a href="glossary/orf-open-reading-frame.html">ORF (Open Reading Frame)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522ORF (Open Reading Frame)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A DNA sequence that possesses a start <a href="glossary.html#codon" title="Three base pairs in either DNA or RNA that code for an amino acid (or stop translation)." class="lexicon-term active">codon</a> and a large window of sequence with no stop codon that could potentially code for a protein.</p>
         </dd>
<dt>
<a id="Ortholog"></a>
  <a href="glossary/ortholog.html">Ortholog</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Ortholog%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Orthologs are genes derived from a common ancestor through vertical descent (or speciation) and can be thought of as the direct evolutionary counterpart. In contrast, paralogues are genes within the same genome that have evolved by duplication.</p>
         </dd>
  </dl>
                  <a id="letter_p"></a>
  <h2 class="lexicon-letter">P</h2>
  <dl>
    <dt>
<a id="PAR_Pseudoautosomal_region"></a>
  <a href="glossary/par-pseudoautosomal-region.html">PAR (Pseudoautosomal region)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522PAR (Pseudoautosomal region)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Small regions of <a href="glossary.html#Sequence_identity" title="A measure of how similar two sequences are, specifically, what percent of amino acids are the same in type and position between the two sequences." class="lexicon-term active">sequence identity</a> located at the tips of the short and the long arms of the X and Y chromosomes where recombination and genetic exchange take place. Genes within the pseudoautosomal region are not sex linked. The Genome Reference Consortium defines two PARs for the human genome <a href="glossary.html#Assembly" title="When the genome or transcriptome of a species is sequenced, typically many short random fragments are sequenced and reassembled by a computer algorithm into longer contiguous sequences called contigs. Genomic contigs may be assembled into longer sequences called scaffolds and sometimes, if the depth of sequencing is high enough, there may be enough information to assemble most of the scaffolds into chromosomes." class="lexicon-term active">assembly</a>. The first pseudoautosomal region, PAR1, is located at the tip of the short arm and consists entirely of N's. The second pseudoautosomal region, PAR2, is located at the tip of the long arm. In the Ensembl human database, DNA for the complete X chromosome is stored and annotated. Only the two unique regions of the Y chromosome are stored and annotated. We are able to represent the complete Y chromosome by filling the 'gaps' with the two PAR regions from the X chromosome. This is done on-the-fly using our assembly_exceptions table. Please note that when using the API, SliceAdaptor by default will fetch only the unique regions of the genome. This means that the PARs on chromosome X will be fetched but only the unique regions on Y will be fetched. To fetch the full length of the Y chromosome using the SliceAdaptor, set the 4th argument to '1' as shown: my $slices = $<a href="glossary.html#Slice" title="The term &quot;slice&quot; in Ensembl refers to a length of DNA sequence. A slice can be any length, from one base long to the entire length of a chromosome. A slice is defined as follows:    &#039;coord_system_name:coord_system_version:seq_region_name:start:end:strand&#039;    eg. &#039;chromosome:GRCm38:X:1000:2000:1&#039;" class="lexicon-term active">slice</a>_adaptor-&gt;fetch_all( '<a href="glossary.html#Toplevel" title="The largest continuous sequence for an organism. The official technical definition for toplevel sequences are &#039;sequence regions in the genome assembly that are not a component of another sequence region&#039;. For example, when a genome is assembled into chromosomes, toplevel sequences will be chromosomes and unplaced scaffolds. If a genome has only been assembled into scaffolds, then toplevel sequences are scaffolds and unplaced contigs." class="lexicon-term active">toplevel</a>', 'GRCh37', 0, 1 );</p>
         </dd>
<dt>
<a id="Paralog"></a>
  <a href="glossary/paralog.html">Paralog</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Paralog%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A sequence that has evolved by duplication: "<em>Anopheles albimanus</em> has four paralogs of gene X while <em>Anopheles gambiae</em> has only one copy"</p>
         </dd>
<dt>
<a id="PDB_Protein_Data_Bank"></a>
  <a href="glossary/pdb-protein-data-bank.html">PDB (Protein Data Bank)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522PDB (Protein Data Bank)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A repository for 3-D biological macromolecular structure data. PDB archives protein structures deduced from crystallography and nuclear magnetic reasonance (NMR) experiments on protein structures. The Protein Data Bank (PDB) is operated by Rutgers, The State University of New Jersey; the San Diego Supercomputer Center at the University of California, San Diego; and the Center for Advanced Research in Biotechnology of the National Institute of Standards and Technology -- three members of the Research Collaboratory for Structural Bioinformatics (RCSB). The RCSB PDB is supported by funds from the National Science Foundation, the Department of Energy, and the National Institutes of Health.</p>
         </dd>
<dt>
<a id="Pfam"></a>
  <a href="glossary/pfam.html">Pfam</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Pfam%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p><a href="glossary.html#Pfam" title="Pfam is a large collection of multiple sequence alignments and hidden Markov models covering many common protein domains and families. Pfam can be used to view the domain organization of proteins, to view multiple alignments, protein domain architectures, protein structures, and species distributions." class="lexicon-term active">Pfam</a> is a large collection of multiple sequence alignments and hidden Markov models covering many common protein domains and families. Pfam can be used to view the <a href="glossary.html#Domain" title="A region of special biological interest within a single protein sequence. However, a domain may also be defined as a region within the three-dimensional structure of a protein that may encompass regions of several distinct protein sequences that accomplishes a specific function. A domain class is a group of domains that share a common set of well-defined properties or characteristics." class="lexicon-term active">domain</a> organization of proteins, to view multiple alignments, protein domain architectures, protein structures, and species distributions.</p>
         </dd>
<dt>
<a id="Pmatch"></a>
  <a href="glossary/pmatch.html">Pmatch</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Pmatch%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p><a href="glossary.html#Pmatch" title="Pmatch is a fast, exact matching program for aligning protein sequences with either protein or DNA sequence." class="lexicon-term active">Pmatch</a> is a fast, exact matching program for aligning protein sequences with either protein or DNA sequence.</p>
         </dd>
<dt>
<a id="Polymorphic_pseudogene"></a>
  <a href="glossary/polymorphic-pseudogene.html">Polymorphic pseudogene</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Polymorphic pseudogene%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p><a href="glossary.html#Pseudogene" title="A noncoding sequence similar to an active protein. A sequence that closely resembles a known functional gene, at another locus within a genome, that is non-functional as a consequence of (usually several) mutations that prevent either its transcription or translation (or both). In general, pseudogenes result from either reverse transcription of a transcript of their &quot;normal&quot; paralog (SO:0000043) (in which case the pseudogene typically lacks introns and includes a poly(A) tail) or from recombination (SO:0000044) (in which case the pseudogene is typically a tandem duplication of its &quot;normal&quot; paralog). Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000336" class="lexicon-term active">Pseudogene</a> owing to a polymorphism in the reference genome, translated in other individuals/<a href="glossary.html#Haplotypes" title="A set of genes or markers on one chromosome that are inherited together. Often refers to SNPs that are closely linked (i.e. have a high linkage disequilibrium (LD) value, and are inherited together.) In Region in Detail, the haplotype regions are coloured with a red background." class="lexicon-term active">haplotypes</a>/strains.</p>
         </dd>
<dt>
<a id="Pre-release_site"></a>
  <a href="glossary/pre-release-site.html">Pre-release site</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Pre-release site%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Initial annotations of upcoming Ensembl genomes, usually without gene predictions or validation, are regularly made available on the <a href="glossary.html#Pre-release_site" title="Initial annotations of upcoming Ensembl genomes, usually without gene predictions or validation, are regularly made available on the pre-release site, pre.ensembl.org" class="lexicon-term active">pre-release site</a>, pre.ensembl.org</p>
         </dd>
<dt>
<a id="pre_miRNA"></a>
  <a href="glossary/premirna.html">pre_miRNA</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522pre_miRNA%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
        The 60-70 nucleotide region remain after Drosha processing of the primary transcript, that folds back upon itself to form a hairpin structure. <a href="http://www.sequenceontology.org/browser/current_svn/term/SO:0001244">Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0001244</a>        </dd>
<dt>
<a id="Prints"></a>
  <a href="glossary/prints.html">Prints</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Prints%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>The <a href="glossary.html#Prints" title="The PRINTS protein fingerprint database is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of a SwissProt/TrEMBL composite. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs, full diagnostic potency deriving from the mutual context provided by motif neighbors." class="lexicon-term active">PRINTS</a> protein fingerprint database is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of a SwissProt/TrEMBL composite. Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space. Fingerprints can encode protein folds and functionalities more flexibly and powerfully than can single motifs, full diagnostic potency deriving from the mutual context provided by <a href="glossary.html#Motif" title="A conserved region of sequence with a specific function/ structure." class="lexicon-term active">motif</a> neighbors.</p>
         </dd>
<dt>
<a id="Processed_pseudogene"></a>
  <a href="glossary/processed-pseudogene.html">Processed pseudogene</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Processed pseudogene%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Noncoding <a href="glossary.html#Pseudogene" title="A noncoding sequence similar to an active protein. A sequence that closely resembles a known functional gene, at another locus within a genome, that is non-functional as a consequence of (usually several) mutations that prevent either its transcription or translation (or both). In general, pseudogenes result from either reverse transcription of a transcript of their &quot;normal&quot; paralog (SO:0000043) (in which case the pseudogene typically lacks introns and includes a poly(A) tail) or from recombination (SO:0000044) (in which case the pseudogene is typically a tandem duplication of its &quot;normal&quot; paralog). Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000336" class="lexicon-term active">pseudogene</a> produced by integration of a reverse transcribed mRNA into the genome.</p>
         </dd>
<dt>
<a id="Processed_transcript"></a>
  <a href="glossary/processed-transcript.html">Processed transcript</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Processed transcript%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Noncoding <a href="glossary.html#Transcript" title="Nucleotide sequence resulting from the transcription of the genomic DNA to mRNA. One gene can have different transcripts or splice variants resulting from the alternative splicing of different exons in genes." class="lexicon-term active">transcript</a> that does not contain an open reading frame (ORF).</p>
         </dd>
<dt>
<a id="Projected_gene_or_known_by_projection"></a>
  <a href="glossary/projected-gene-or-known-byprojection.html">Projected gene (or known by_projection)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Projected gene (or known by_projection)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A projected Ensembl gene has only one or more novel transcripts annotated, and has a <a href="glossary.html#Known_gene" title="A known gene is an Ensembl gene for which at least one known transcript has been annotated." class="lexicon-term active">known gene</a> from human or mouse as an orthologue. The gene symbol and description are projected from the human or mouse orthologue.</p>
         </dd>
<dt>
<a id="Prosite"></a>
  <a href="glossary/prosite.html">Prosite</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Prosite%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p><a href="glossary.html#Prosite" title="PROSITE is a database of protein families and domains run by the (Expert Protein Analysis System (ExPASy) proteomics server of the Swiss Institute of Bioinformatics (SIB). It consists of biologically significant sites, patterns and profiles that help to reliably identify to which known protein family (if any) a new sequence belongs." class="lexicon-term active">PROSITE</a> is a database of protein families and domains run by the (Expert Protein Analysis System (ExPASy) proteomics server of the Swiss Institute of Bioinformatics (SIB). It consists of biologically significant sites, patterns and profiles that help to reliably identify to which known protein family (if any) a new sequence belongs.</p>
         </dd>
<dt>
<a id="Protein_coding"></a>
  <a href="glossary/protein-coding.html">Protein coding</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Protein coding%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A <a href="glossary.html#Protein_coding" title="A protein coding transcript is a spliced mRNA that leads to a protein product." class="lexicon-term active">protein coding</a> <a href="glossary.html#Transcript" title="Nucleotide sequence resulting from the transcription of the genomic DNA to mRNA. One gene can have different transcripts or splice variants resulting from the alternative splicing of different exons in genes." class="lexicon-term active">transcript</a> is a spliced mRNA that leads to a protein product.</p>
         </dd>
<dt>
<a id="Protein_ID"></a>
  <a href="glossary/protein-id.html">Protein ID</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Protein ID%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Ensembl protein IDs are unique for differing translations.</p>
         </dd>
<dt>
<a id="Pseudogene"></a>
  <a href="glossary/pseudogene.html">Pseudogene</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Pseudogene%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A noncoding sequence similar to an active protein. A sequence that closely resembles a known functional gene, at another locus within a genome, that is non-functional as a consequence of (usually several) mutations that prevent either its transcription or translation (or both). In general, pseudogenes result from either reverse transcription of a <a href="glossary.html#Transcript" title="Nucleotide sequence resulting from the transcription of the genomic DNA to mRNA. One gene can have different transcripts or splice variants resulting from the alternative splicing of different exons in genes." class="lexicon-term active">transcript</a> of their "normal" <a href="glossary.html#Paralog" title="A sequence that has evolved by duplication: &quot;Anopheles albimanus has four paralogs of gene X while Anopheles gambiae has only one copy&quot;" class="lexicon-term active">paralog</a> (SO:0000043) (in which case the <a href="glossary.html#Pseudogene" title="A noncoding sequence similar to an active protein. A sequence that closely resembles a known functional gene, at another locus within a genome, that is non-functional as a consequence of (usually several) mutations that prevent either its transcription or translation (or both). In general, pseudogenes result from either reverse transcription of a transcript of their &quot;normal&quot; paralog (SO:0000043) (in which case the pseudogene typically lacks introns and includes a poly(A) tail) or from recombination (SO:0000044) (in which case the pseudogene is typically a tandem duplication of its &quot;normal&quot; paralog). Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000336" class="lexicon-term active">pseudogene</a> typically lacks introns and includes a poly(A) tail) or from recombination (SO:0000044) (in which case the pseudogene is typically a tandem duplication of its "normal" paralog). <a href="http://www.sequenceontology.org/browser/current_svn/term/SO:0000336">Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000336</a></p>
         </dd>
  </dl>
                  <a id="letter_q"></a>
  <h2 class="lexicon-letter">Q</h2>
  <dl>
    <dt>
<a id="QTL_Quantitative_Trait_Locus"></a>
  <a href="glossary/qtl-quantitative-trait-locus.html">QTL (Quantitative Trait Locus)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522QTL (Quantitative Trait Locus)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Genetic loci where allelic variation is associated with variation in a quantitative trait (e.g. blood pressure). The presence of QTL is inferred from genetic mapping. Total variation is partitioned into components linked to a number of discrete, mapped chromosome markers described by statistical association to quantitative variation in a particular phenotypic trait that is thought to be controlled by the cumulative action of alleles at multiple loci.</p>
         </dd>
<dt>
<a id="Query_id"></a>
  <a href="glossary/query-id.html">Query %id</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Query %25id%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p><a href="glossary.html#Query_id" title="Query %id indicates the percentage of the query sequence matching the target sequence." class="lexicon-term active">Query %id</a> indicates the percentage of the query sequence matching the target sequence.</p>
         </dd>
  </dl>
                  <a id="letter_r"></a>
  <h2 class="lexicon-letter">R</h2>
  <dl>
    <dt>
<a id="Reference_SNP_Reference_Single_Nucleotide_Polymorphism"></a>
  <a href="glossary/reference-snp-reference-single-nucleotide-polymorphism.html">Reference SNP (Reference Single Nucleotide Polymorphism)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Reference SNP (Reference Single Nucleotide Polymorphism)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A SNP assigned to eliminate redundancy in the NCBI <a href="glossary.html#dbSNP" title="    The Single Nucleotide Polymorphism database (dbSNP) is a public-domain archive for a broad collection of simple (short) genetic polymorphisms. This collection of polymorphisms is maintained by NCBI and includes single-base nucleotide substitutions (also known as single nucleotide polymorphisms or SNPs), small-scale multi-base deletions or insertions (also called deletion insertion polymorphisms, indels or DIPs), and retroposable element insertions and microsatellite repeat variations (also called short tandem repeats or STRs). See more details here." class="lexicon-term active">dbSNP</a> database. All SNPs submitted at the position of a reference SNP are given the reference SNP identifier (a number preceded by 'rs').</p>
         </dd>
<dt>
<a id="RefSeq"></a>
  <a href="glossary/refseq.html">RefSeq</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522RefSeq%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>NCBI's Reference Sequences (<a href="glossary.html#RefSeq" title="NCBI&#039;s Reference Sequences (RefSeq) database is a curated database of Genbank&#039;s genomes, mRNAs and proteins. RefSeq attempts to provide a comprehensive, integrated, non-redundant set of sequences, including genomic DNA, tRNA, and protein products, providing a stable reference for gene identification and characterization, mutation analysis, expression studies, polymorphism discovery, and comparative analyses." class="lexicon-term active">RefSeq</a>) database is a curated database of Genbank's genomes, mRNAs and proteins. RefSeq attempts to provide a comprehensive, integrated, non-redundant set of sequences, including genomic DNA, <a href="glossary.html#tRNA" title="Transfer RNA. These are identified using tRNAscan. Transfer RNA (tRNA) molecules are approximately 80 nucleotides in length. Their secondary structure includes four short double-helical elements and three loops (D, anti-codon, and T loops). Further hydrogen bonds mediate the characteristic L-shaped molecular structure. Transfer RNAs have two regions of fundamental functional importance: the anti-codon, which is responsible for specific mRNA codon recognition, and the 3&#039; end, to which the tRNA&#039;s corresponding amino acid is attached (by aminoacyl-tRNA synthetases). Transfer RNAs cope with the degeneracy of the genetic code in two manners: having more than one tRNA (with a specific anti-codon) for a particular amino acid; and &#039;wobble&#039; base-pairing, i.e. permitting non-standard base-pairing at the 3rd anti-codon position. Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000253" class="lexicon-term active">tRNA</a>, and protein products, providing a stable reference for gene identification and characterization, <a href="glossary.html#Mutation" title="A modification (insertion, deletion, or alteration) in the genomic or amino acid sequence." class="lexicon-term active">mutation</a> analysis, expression studies, polymorphism discovery, and comparative analyses.</p>
         </dd>
<dt>
<a id="repeat"></a>
  <a href="glossary/repeat.html">repeat</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522repeat%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Repetitive DNA in which the same sequence occurs multiple times.</p>
         </dd>
<dt>
<a id="Repeat_Masking"></a>
  <a href="glossary/repeat-masking.html">Repeat Masking</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Repeat Masking%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>The method by which repeated sequences and low-complexity regions are hidden, usually used in searches by <a href="glossary.html#Alignment" title="A comparison between two or more sequences by matching identical and/or similar residues and assigning a score to the match." class="lexicon-term active">alignment</a> and homology-searching programs.</p>
         </dd>
<dt>
<a id="RepeatMasker"></a>
  <a href="glossary/repeatmasker.html">RepeatMasker</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522RepeatMasker%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p><a href="glossary.html#RepeatMasker" title="RepeatMasker (AFA Smit &amp; P Green) is a standard software tool used in computational genomics to identify repetitive elements and low-complexity sequences." class="lexicon-term active">RepeatMasker</a> (AFA Smit &amp; P Green) is a standard software tool used in computational genomics to identify repetitive elements and low-complexity sequences.</p>
         </dd>
<dt>
<a id="Retained_intron"></a>
  <a href="glossary/retained-intron.html">Retained intron</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Retained intron%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Noncoding <a href="glossary.html#Transcript" title="Nucleotide sequence resulting from the transcription of the genomic DNA to mRNA. One gene can have different transcripts or splice variants resulting from the alternative splicing of different exons in genes." class="lexicon-term active">transcript</a> containing intronic sequence.</p>
         </dd>
<dt>
<a id="Retrotransposed"></a>
  <a href="glossary/retrotransposed.html">Retrotransposed</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Retrotransposed%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A noncoding <a href="glossary.html#Pseudogene" title="A noncoding sequence similar to an active protein. A sequence that closely resembles a known functional gene, at another locus within a genome, that is non-functional as a consequence of (usually several) mutations that prevent either its transcription or translation (or both). In general, pseudogenes result from either reverse transcription of a transcript of their &quot;normal&quot; paralog (SO:0000043) (in which case the pseudogene typically lacks introns and includes a poly(A) tail) or from recombination (SO:0000044) (in which case the pseudogene is typically a tandem duplication of its &quot;normal&quot; paralog). Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000336" class="lexicon-term active">pseudogene</a> produced by integration of a reverse transcribed mRNA into the genome.</p>
         </dd>
<dt>
<a id="RH_map_Radiation_Hybrid_map"></a>
  <a href="glossary/rh-map-radiation-hybrid-map.html">RH map (Radiation Hybrid map)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522RH map (Radiation Hybrid map)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Technique for identifying landmarks (STS) every 100 kb in the human genome, the ordering is relative to the frequency with which they are separated by radiation-induced breaks. The frequency is assayed by analysing a panel of human-hamster hybrid cell lines.</p>
         </dd>
<dt>
<a id="ribozyme"></a>
  <a href="glossary/ribozyme.html">ribozyme</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522ribozyme%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
        An RNA with catalytic activity. <a href="http://www.sequenceontology.org/browser/current_svn/term/SO:0000374">Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000374</a>        </dd>
<dt>
<a id="RNase_MRP_RNA"></a>
  <a href="glossary/rnasemrprna.html">RNase_MRP_RNA</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522RNase_MRP_RNA%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
        The RNA molecule essential for the catalytic activity of RNase MRP, an enzymatically active ribonucleoprotein with two distinct roles in eukaryotes. In mitochondria it plays a direct role in the initiation of mitochondrial DNA replication. In the nucleus it is involved in precursor rRNA processing, where it cleaves the internal transcribed spacer 1 between 18S and 5.8S rRNAs. <a href="http://www.sequenceontology.org/browser/current_svn/term/SO:0000385">Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000385</a>        </dd>
<dt>
<a id="RNase_P_RNA"></a>
  <a href="glossary/rnaseprna.html">RNase_P_RNA</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522RNase_P_RNA%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
        The RNA component of Ribonuclease P (RNase P), a ubiquitous endoribonuclease, found in archaea, bacteria and eukarya as well as chloroplasts and mitochondria. Its best characterized activity is the generation of mature 5 prime ends of tRNAs by cleaving the 5 prime leader elements of precursor-tRNAs. Cellular RNase Ps are ribonucleoproteins. RNA from bacterial RNase Ps retains its catalytic activity in the absence of the protein subunit, i.e. it is a ribozyme. Isolated eukaryotic and archaeal RNase P RNA has not been shown to retain its catalytic function, but is still essential for the catalytic activity of the holoenzyme. Although the archaeal and eukaryotic holoenzymes have a much greater protein content than the bacterial ones, the RNA cores from all the three lineages are homologous. Helices corresponding to P1, P2, P3, P4, and P10/11 are common to all cellular RNase P RNAs. Yet, there is considerable sequence variation, particularly among the eukaryotic RNAs. <a href="http://www.sequenceontology.org/browser/current_svn/term/SO:0000386">Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000386</a>        </dd>
<dt>
<a id="rRNA"></a>
  <a href="glossary/rrna.html">rRNA</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522rRNA%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Ribosomal RNA. RNA that comprises part of a ribosome, and that can provide both structural scaffolding and catalytic activity. <a href="http://www.sequenceontology.org/browser/current_svn/term/SO:0000252">Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000252</a></p>
         </dd>
<dt>
<a id="rRNA_pseudogene"></a>
  <a href="glossary/rrna-pseudogene.html">rRNA pseudogene</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522rRNA pseudogene%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Ribosomal RNA <a href="glossary.html#Pseudogene" title="A noncoding sequence similar to an active protein. A sequence that closely resembles a known functional gene, at another locus within a genome, that is non-functional as a consequence of (usually several) mutations that prevent either its transcription or translation (or both). In general, pseudogenes result from either reverse transcription of a transcript of their &quot;normal&quot; paralog (SO:0000043) (in which case the pseudogene typically lacks introns and includes a poly(A) tail) or from recombination (SO:0000044) (in which case the pseudogene is typically a tandem duplication of its &quot;normal&quot; paralog). Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000336" class="lexicon-term active">pseudogene</a>.</p>
         </dd>
  </dl>
                  <a id="letter_s"></a>
  <h2 class="lexicon-letter">S</h2>
  <dl>
    <dt>
<a id="SARA_Same_As_Reference_Assembly"></a>
  <a href="glossary/sara-same-reference-assembly.html">SARA (Same As Reference Assembly)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522SARA (Same As Reference Assembly)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>An acronym used to indicate a <a href="glossary.html#SNP_Single_Nucleotide_Polymorphism" title="SNPs are common variations that occur in DNA with a 0.1% frequency. Ensembl displays SNPs obtained from dbSNP, (the SNP repository maintained by NCBI; The Human Genic Bi-Allelic Sequences Database (HGVBase) and The SNP Consortium Ltd.(TSC)." class="lexicon-term active">SNP (single nucleotide polymorphism)</a> that has the same sequence as the strain used in the <a href="glossary.html#Assembly" title="When the genome or transcriptome of a species is sequenced, typically many short random fragments are sequenced and reassembled by a computer algorithm into longer contiguous sequences called contigs. Genomic contigs may be assembled into longer sequences called scaffolds and sometimes, if the depth of sequencing is high enough, there may be enough information to assemble most of the scaffolds into chromosomes." class="lexicon-term active">assembly</a>.</p>
         </dd>
<dt>
<a id="Scaffold"></a>
  <a href="glossary/scaffold.html">Scaffold</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Scaffold%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p><a href="glossary.html#supercontigs" title="Assemblies consist of sequence contigs combined into scaffolds, also known as supercontigs. Supercontigs are combined and ordered according to their orientation and linking information provided by mated sequences from the ends of genomic sub-clones. For some species, supercontigs are combined into ultracontigs, in which neighboring supercontigs are organized into their proper order and orientation using linking information provided by the physical map of BAC clones independently assembled using restriction fragment patterns and the FPC program." class="lexicon-term active">Supercontigs</a> or scaffolds are sets of ordered, oriented contigs. They are longer sequences than contigs, but shorter than full chromosomes.</p>
         </dd>
<dt>
<a id="scaRNA"></a>
  <a href="glossary/scarna.html">scaRNA</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522scaRNA%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
        A ncRNA, specific to the Cajal body, that has been demonstrated to function as a guide RNA in the site-specific synthesis of 2'-O-ribose-methylated nucleotides and pseudouridines in the RNA polymerase II-transcribed U1, U2, U4 and U5 spliceosomal small nuclear RNAs (snRNAs). <a href="http://www.sequenceontology.org/browser/current_svn/term/SO:0002095">Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0002095</a>        </dd>
<dt>
<a id="ScRNA"></a>
  <a href="glossary/scrna.html">ScRNA</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522ScRNA%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Small cytoplasmic RNA.</p>
         </dd>
<dt>
<a id="ScRNA_pseudogene"></a>
  <a href="glossary/scrna-pseudogene.html">ScRNA pseudogene</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522ScRNA pseudogene%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Small cytoplasmic RNA <a href="glossary.html#Pseudogene" title="A noncoding sequence similar to an active protein. A sequence that closely resembles a known functional gene, at another locus within a genome, that is non-functional as a consequence of (usually several) mutations that prevent either its transcription or translation (or both). In general, pseudogenes result from either reverse transcription of a transcript of their &quot;normal&quot; paralog (SO:0000043) (in which case the pseudogene typically lacks introns and includes a poly(A) tail) or from recombination (SO:0000044) (in which case the pseudogene is typically a tandem duplication of its &quot;normal&quot; paralog). Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000336" class="lexicon-term active">pseudogene</a>.</p>
         </dd>
<dt>
<a id="SEG"></a>
  <a href="glossary/seg.html">SEG</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522SEG%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p><a href="glossary.html#SEG" title="Seg divides sequences into contrasting segments of low-complexity and high-complexity. Low-complexity segments defined by the algorithm represent &quot;simple sequences&quot; or &quot;compositionally-biased regions&quot;. Segment lengths and the number of segments per sequence are determined automatically by the algorithm." class="lexicon-term active">Seg</a> divides sequences into contrasting segments of low-complexity and high-complexity. Low-complexity segments defined by the <a href="glossary.html#Algorithm" title="A sequence of computational tasks or actions that carry out a specific function." class="lexicon-term active">algorithm</a> represent "simple sequences" or "compositionally-biased regions". Segment lengths and the number of segments per sequence are determined automatically by the algorithm.</p>
         </dd>
<dt>
<a id="sense_intronic"></a>
  <a href="glossary/senseintronic.html">sense_intronic</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522sense_intronic%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
        A non-coding transcript found within an intron of a coding or non-coding gene, with no overlap of exonic sequence. <a href="http://www.sequenceontology.org/browser/current_svn/term/SO:0002131">Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0002131</a>        </dd>
<dt>
<a id="Sequence_identity"></a>
  <a href="glossary/sequence-identity.html">Sequence identity</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Sequence identity%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A measure of how similar two sequences are, specifically, what percent of amino acids are the same in type and position between the two sequences.</p>
         </dd>
<dt>
<a id="Shotgun_method"></a>
  <a href="glossary/shotgun-method.html">Shotgun method</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Shotgun method%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>(also whole genome shotgun) Semi-automated sequencing method that involves randomly sequenced cloned pieces of the genome (size selected, sually 2, 10, 50 and 150 kb), with no prior knowledge their location. The clones are then sequenced from both ends. The two ends of the same <a href="glossary.html#Clone" title="A segment of DNA that has been inserted into a vector molecule, such as a plasmid, and then replicated to form many identical copies." class="lexicon-term active">clone</a> are referred to as mate pairs. The distance between two "mate pairs" can be inferred if the library size is known and has a narrow window of deviation. This approach can be contrasted with "directed" strategies, in which pieces of DNA from known chromosomal locations are sequenced.</p>
         </dd>
<dt>
<a id="Shotgun_sequencing"></a>
  <a href="glossary/shotgun-sequencing.html">Shotgun sequencing</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Shotgun sequencing%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A method in which small, random DNA sequences are generated that overlap. The fragments are sequenced and the full, connected sequence determined through the overlaps.</p>
         </dd>
<dt>
<a id="SIFT"></a>
  <a href="glossary/sift.html">SIFT</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522SIFT%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A tool which predicts the variation effect on protein function based on sequence homology and the physico-chemical <a href="glossary.html#Similarity" title="How well one sequence matches another determined by calculation by an alignment program of identical and conserved residues." class="lexicon-term active">similarity</a> between the alternate amino acids. See the <a href="glossary.html#SIFT" title="A tool which predicts the variation effect on protein function based on sequence homology and the physico-chemical similarity between the alternate amino acids. See the SIFT website for more information." class="lexicon-term active">SIFT</a> website for more information.</p>
         </dd>
<dt>
<a id="SignalP"></a>
  <a href="glossary/signalp.html">SignalP</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522SignalP%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>The <a href="glossary.html#SignalP" title="The SignalP application predicts the presence and location of signal peptide cleavage sites in amino acid sequences from different organisms: Gram-positive prokaryotes, Gram-negative prokaryotes, and eukaryotes. The method incorporates a prediction of cleavage sites and a signal peptide/non-signal peptide prediction based on a combination of several artificial neural networks. Signal peptides indicate a protein that will be secreted. Prediction of signal peptides is quite accurate however care must be exercised and these regions should be verified by other means. (Henrik Nielsen, Jacob Engelbrecht, Sren Brunak and Gunnar von Heijne. Identification of prokaryotic and eukaryotic signal peptides and prediction of their cleavage sites. Protein Engineering 10, 1-6 (1997)" class="lexicon-term active">SignalP</a> application predicts the presence and location of signal peptide cleavage sites in amino acid sequences from different organisms: Gram-positive prokaryotes, Gram-negative prokaryotes, and eukaryotes. The method incorporates a prediction of cleavage sites and a signal peptide/non-signal peptide prediction based on a combination of several artificial neural networks. Signal peptides indicate a protein that will be secreted. Prediction of signal peptides is quite accurate however care must be exercised and these regions should be verified by other means. (Henrik Nielsen, Jacob Engelbrecht, Sren Brunak and Gunnar von Heijne. Identification of prokaryotic and eukaryotic signal peptides and prediction of their cleavage sites. Protein Engineering 10, 1-6 (1997)</p>
         </dd>
<dt>
<a id="Similarity"></a>
  <a href="glossary/similarity.html">Similarity</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Similarity%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>How well one sequence matches another determined by calculation by an <a href="glossary.html#Alignment" title="A comparison between two or more sequences by matching identical and/or similar residues and assigning a score to the match." class="lexicon-term active">alignment</a> program of identical and conserved residues.</p>
         </dd>
<dt>
<a id="Slice"></a>
  <a href="glossary/slice.html">Slice</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Slice%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>The term "<a href="glossary.html#Slice" title="The term &quot;slice&quot; in Ensembl refers to a length of DNA sequence. A slice can be any length, from one base long to the entire length of a chromosome. A slice is defined as follows:    &#039;coord_system_name:coord_system_version:seq_region_name:start:end:strand&#039;    eg. &#039;chromosome:GRCm38:X:1000:2000:1&#039;" class="lexicon-term active">slice</a>" in Ensembl refers to a length of DNA sequence. A slice can be any length, from one base long to the entire length of a chromosome. A slice is defined as follows:    'coord_system_name:coord_system_version:seq_region_name:start:end:strand'    eg. 'chromosome:GRCm38:X:1000:2000:1'</p>
         </dd>
<dt>
<a id="SNAP"></a>
  <a href="glossary/snap.html">SNAP</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522SNAP%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>1. (Synonymous/Non-synonymous Analysis Program) A program which calculates synonymous and non-synonymous substitution rates based on a set of <a href="glossary.html#codon" title="Three base pairs in either DNA or RNA that code for an amino acid (or stop translation)." class="lexicon-term active">codon</a>-aligned nucleotide sequences, based on the method of Nei and Gojobori, incorporating a statistic developed in Ota and Nei.       2. An ab initio gene prediction program developed by Ian Korf that models <a href="glossary.html#Protein_coding" title="A protein coding transcript is a spliced mRNA that leads to a protein product." class="lexicon-term active">protein coding</a> sequences in genomic DNA by means of hidden Markov models.</p>
         </dd>
<dt>
<a id="snoRNA"></a>
  <a href="glossary/snorna.html">snoRNA</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522snoRNA%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Small nucleolar RNA, involved in modifications of other RNAs. A <a href="glossary.html#snoRNA" title="Small nucleolar RNA, involved in modifications of other RNAs. A snoRNA (small nucleolar RNA) is any one of a class of small RNAs that are associated with the eukaryotic nucleus as components of small nucleolar ribonucleoproteins. They participate in the processing or modifications of many RNAs, mostly ribosomal RNAs (rRNAs) though snoRNAs are also known to target other classes of RNA, including spliceosomal RNAs, tRNAs, and mRNAs via a stretch of sequence that is complementary to a sequence in the targeted RNA. Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000275" class="lexicon-term active">snoRNA</a> (small nucleolar RNA) is any one of a class of small RNAs that are associated with the eukaryotic nucleus as components of small nucleolar ribonucleoproteins. They participate in the processing or modifications of many RNAs, mostly ribosomal RNAs (rRNAs) though snoRNAs are also known to target other classes of RNA, including spliceosomal RNAs, tRNAs, and mRNAs via a stretch of sequence that is complementary to a sequence in the targeted RNA. <a href="http://www.sequenceontology.org/browser/current_svn/term/SO:0000275">Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000275</a></p>
         </dd>
<dt>
<a id="SnoRNA_pseudogene"></a>
  <a href="glossary/snorna-pseudogene.html">SnoRNA pseudogene</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522SnoRNA pseudogene%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Small nucleolar RNA <a href="glossary.html#Pseudogene" title="A noncoding sequence similar to an active protein. A sequence that closely resembles a known functional gene, at another locus within a genome, that is non-functional as a consequence of (usually several) mutations that prevent either its transcription or translation (or both). In general, pseudogenes result from either reverse transcription of a transcript of their &quot;normal&quot; paralog (SO:0000043) (in which case the pseudogene typically lacks introns and includes a poly(A) tail) or from recombination (SO:0000044) (in which case the pseudogene is typically a tandem duplication of its &quot;normal&quot; paralog). Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000336" class="lexicon-term active">pseudogene</a>, involved in modifications of other RNAs.</p>
         </dd>
<dt>
<a id="SNP_Single_Nucleotide_Polymorphism"></a>
  <a href="glossary/snp-single-nucleotide-polymorphism.html">SNP (Single Nucleotide Polymorphism)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522SNP (Single Nucleotide Polymorphism)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>SNPs are common variations that occur in DNA with a 0.1% frequency. Ensembl displays SNPs obtained from <a href="glossary.html#dbSNP" title="    The Single Nucleotide Polymorphism database (dbSNP) is a public-domain archive for a broad collection of simple (short) genetic polymorphisms. This collection of polymorphisms is maintained by NCBI and includes single-base nucleotide substitutions (also known as single nucleotide polymorphisms or SNPs), small-scale multi-base deletions or insertions (also called deletion insertion polymorphisms, indels or DIPs), and retroposable element insertions and microsatellite repeat variations (also called short tandem repeats or STRs). See more details here." class="lexicon-term active">dbSNP</a>, (the SNP repository maintained by NCBI; The Human Genic Bi-Allelic Sequences Database (HGVBase) and The SNP Consortium Ltd.(TSC).</p>
         </dd>
<dt>
<a id="snRNA"></a>
  <a href="glossary/snrna.html">snRNA</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522snRNA%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A small nuclear RNA molecule involved in pre-mRNA splicing and processing. <a href="http://www.sequenceontology.org/browser/current_svn/term/SO:0000274">Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000274</a></p>
         </dd>
<dt>
<a id="SnRNA_pseudogene"></a>
  <a href="glossary/snrna-pseudogene.html">SnRNA pseudogene</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522SnRNA pseudogene%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Small nuclear RNA <a href="glossary.html#Pseudogene" title="A noncoding sequence similar to an active protein. A sequence that closely resembles a known functional gene, at another locus within a genome, that is non-functional as a consequence of (usually several) mutations that prevent either its transcription or translation (or both). In general, pseudogenes result from either reverse transcription of a transcript of their &quot;normal&quot; paralog (SO:0000043) (in which case the pseudogene typically lacks introns and includes a poly(A) tail) or from recombination (SO:0000044) (in which case the pseudogene is typically a tandem duplication of its &quot;normal&quot; paralog). Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000336" class="lexicon-term active">pseudogene</a>.</p>
         </dd>
<dt>
<a id="SNV"></a>
  <a href="glossary/snv.html">SNV</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522SNV%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A Single Nucleotide Variant (<a href="glossary.html#SNV" title="A Single Nucleotide Variant (SNV) is a nucleotide position in genomic DNA at which different sequence alternatives (alleles) exist. SNVs include SNPs and single nucleotide insertions or deletions. See more details here." class="lexicon-term active">SNV</a>) is a nucleotide position in genomic DNA at which different sequence alternatives (alleles) exist. SNVs include SNPs and single nucleotide insertions or deletions. See more details here.</p>
         </dd>
<dt>
<a id="Solexa_sequencing"></a>
  <a href="glossary/solexa-sequencing.html">Solexa sequencing</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Solexa sequencing%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         Type of Next-generation sequencing (NGS).         </dd>
<dt>
<a id="SRP_RNA"></a>
  <a href="glossary/srprna.html">SRP_RNA</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522SRP_RNA%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
        The signal recognition particle (SRP) is a universally conserved ribonucleoprotein. It is involved in the co-translational targeting of proteins to membranes. The eukaryotic SRP consists of a 300-nucleotide 7S RNA and six proteins: SRPs 72, 68, 54, 19, 14, and 9. Archaeal SRP consists of a 7S RNA and homologues of the eukaryotic SRP19 and SRP54 proteins. In most eubacteria, the SRP consists of a 4.5S RNA and the Ffh protein (a homologue of the eukaryotic SRP54 protein). Eukaryotic and archaeal 7S RNAs have very similar secondary structures, with eight helical elements. These fold into the Alu and S domains, separated by a long linker region. Eubacterial SRP is generally a simpler structure, with the M domain of Ffh bound to a region of the 4.5S RNA that corresponds to helix 8 of the eukaryotic and archaeal SRP S domain. Some Gram-positive bacteria (<em>e.g. Bacillus subtilis</em>), however, have a larger SRP RNA that also has an Alu domain. The Alu domain is thought to mediate the peptide chain elongation retardation function of the SRP. The universally conserved helix which interacts with the SRP54/Ffh M domain mediates signal sequence recognition. In eukaryotes and archaea, the SRP19-helix 6 complex is thought to be involved in SRP assembly and stabilizes helix 8 for SRP54 binding. <a href="http://www.sequenceontology.org/browser/current_svn/term/SO:0000590">Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000590</a>        </dd>
<dt>
<a id="SSAHA_Sequence_Search_and_Alignment_by_Hashing_Algorithm"></a>
  <a href="glossary/ssaha-sequence-search-and-alignment-hashing-algorithm.html">SSAHA (Sequence Search and Alignment by Hashing Algorithm)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522SSAHA (Sequence Search and Alignment by Hashing Algorithm)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A search designed to detect exact matches, or nearly exact matches, in DNA or protein databases. The SSAHA search has been optimized for alignments of high percentage identity and display as results the most significant matches for ungapped alignments between sequences. Each exact match in an SSAHA <a href="glossary.html#Alignment" title="A comparison between two or more sequences by matching identical and/or similar residues and assigning a score to the match." class="lexicon-term active">alignment</a> is analogous to finding a high-scoring segment pair in <a href="glossary.html#BLAST" title="BLAST (Basic Local Alignment Search Tool) is a sequence comparison algorithm optimised for speed which is used to search sequence databases for optimal local alignments to a query. (Altschul et al., J Mol Biol 215:403-410; 1990)" class="lexicon-term active">BLAST</a>. A number of consecutive matches on a <a href="glossary.html#Contig" title="A contig is a contiguous stretch of DNA sequence without gaps that has been assembled solely based on direct sequencing information.    Short sequences (reads) from a fragmented genome are compared against one another, and overlapping reads are merged to produce one long sequence. This merging process is iterative: overlapping reads are added to the merged sequence whenever possible and so the merged sequence becomes even longer. When no further reads overlap the long merged sequence, then this sequence - called a contig - has reached its maximum length.    Contig can be used in other contexts: A contig can be the sequence corresponding to only one clone. A contig map shows the regions of a chromosome where contiguous DNA segments overlap." class="lexicon-term active">contig</a> may represent features of a gene such as exons or 5' and 3' untranslated regions, depending on the nature of the query sequence.</p>
         </dd>
<dt>
<a id="Stable_ID_Stable_identifier"></a>
  <a href="glossary/stable-id-stable-identifier.html">Stable ID (Stable identifier)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Stable ID (Stable identifier)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Stable identifiers are defined for a number of features including genes, transcripts, translations, exons. Stable IDs for all species follow the same format:<br /><br />     species stable ID prefix + 6 numbers + <a href="glossary.html#Feature" title="Any annotation on a specific location in the genomic sequence." class="lexicon-term active">feature</a> type suffix<br /><br /> Stable IDs are versioned.</p>
         </dd>
<dt>
<a id="Start_phase"></a>
  <a href="glossary/start-phase.html">Start phase</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Start phase%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>In protein-coding exons, the <a href="glossary.html#Start_phase" title="In protein-coding exons, the start phase is the place where the intron lands inside the codon : 0 between codons, 1 between the 1st and second base, 2 between the second and 3rd base. Exons therefore have a start phase and an end phase, but introns have just one phase." class="lexicon-term active">start phase</a> is the place where the <a href="glossary.html#Intron" title="The part of the genomic sequence that is transcribed and then spliced out of the transcript (mRNA). Noncoding." class="lexicon-term active">intron</a> lands inside the <a href="glossary.html#codon" title="Three base pairs in either DNA or RNA that code for an amino acid (or stop translation)." class="lexicon-term active">codon</a> : 0 between codons, 1 between the 1st and second base, 2 between the second and 3rd base. Exons therefore have a start phase and an <a href="glossary.html#End_phase" title="In protein-coding exons, it is usually the case that end phase = (phase + exon_length)%3 but end_phase could be -1 if the exon is half-coding and its 3 prime end is UTR." class="lexicon-term active">end phase</a>, but introns have just one phase.</p>
         </dd>
<dt>
<a id="STS_markers"></a>
  <a href="glossary/sts-markers.html">STS markers</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522STS markers%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p><a href="glossary.html#STS_markers" title="STS markers are short sequences of genomic DNA that can be uniquely amplified by the polymerase chain reaction (PCR) using a pair of primers. Because each is unique, STSs are often used in linkage and radiation hybrid mapping techniques. STSs serve as landmarks on the physical map of the human genome." class="lexicon-term active">STS markers</a> are short sequences of genomic DNA that can be uniquely amplified by the polymerase chain reaction (PCR) using a pair of primers. Because each is unique, STSs are often used in <a href="glossary.html#Linkage" title="A measure of how often features (genes, specific sequences) on a chromosome are inherited together." class="lexicon-term active">linkage</a> and radiation hybrid mapping techniques. STSs serve as landmarks on the physical map of the human genome.</p>
         </dd>
<dt>
<a id="Supercontig"></a>
  <a href="glossary/supercontig.html">Supercontig</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Supercontig%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p><a href="glossary.html#supercontigs" title="Assemblies consist of sequence contigs combined into scaffolds, also known as supercontigs. Supercontigs are combined and ordered according to their orientation and linking information provided by mated sequences from the ends of genomic sub-clones. For some species, supercontigs are combined into ultracontigs, in which neighboring supercontigs are organized into their proper order and orientation using linking information provided by the physical map of BAC clones independently assembled using restriction fragment patterns and the FPC program." class="lexicon-term active">Supercontigs</a> or scaffolds are sets of ordered, oriented contigs. They are longer sequences than contigs, but shorter than full chromosomes.</p>
         </dd>
<dt>
<a id="supercontigs"></a>
  <a href="glossary/supercontigs.html">supercontigs</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522supercontigs%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Assemblies consist of sequence contigs combined into scaffolds, also known as <a href="glossary.html#supercontigs" title="Assemblies consist of sequence contigs combined into scaffolds, also known as supercontigs. Supercontigs are combined and ordered according to their orientation and linking information provided by mated sequences from the ends of genomic sub-clones. For some species, supercontigs are combined into ultracontigs, in which neighboring supercontigs are organized into their proper order and orientation using linking information provided by the physical map of BAC clones independently assembled using restriction fragment patterns and the FPC program." class="lexicon-term active">supercontigs</a>. Supercontigs are combined and ordered according to their orientation and linking information provided by mated sequences from the ends of genomic sub-clones. For some species, supercontigs are combined into ultracontigs, in which neighboring supercontigs are organized into their proper order and orientation using linking information provided by the physical map of BAC clones independently assembled using restriction fragment patterns and the FPC program.</p>
         </dd>
<dt>
<a id="SV"></a>
  <a href="glossary/sv.html">SV</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522SV%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Structural variation. It is generally defined as a region of DNA approximately 1 kb and larger in size and can include inversions and balanced translocations or genomic imbalances (insertions and deletions), commonly referred to as copy number variants (CNVs). More details here.</p>
         </dd>
<dt>
<a id="Synteny"></a>
  <a href="glossary/synteny.html">Synteny</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Synteny%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>The term <a href="glossary.html#Synteny" title="The term synteny was originally defined to mean that two gene loci share the same chromosome. In a genomic context we refer to syntenic regions if both sequence and gene order is conserved between two (closely related) species." class="lexicon-term active">synteny</a> was originally defined to mean that two gene loci share the same chromosome. In a genomic context we refer to syntenic regions if both sequence and gene order is conserved between two (closely related) species.</p>
         </dd>
  </dl>
                  <a id="letter_t"></a>
  <h2 class="lexicon-letter">T</h2>
  <dl>
    <dt>
<a id="tandem_repeats"></a>
  <a href="glossary/tandem-repeats.html">tandem repeats</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522tandem repeats%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Multiple copies of the same base sequence on a chromosome; used as markers in physical mapping.</p>
         </dd>
<dt>
<a id="Target_id"></a>
  <a href="glossary/target-id.html">Target % id</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Target %25 id%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Target %id indicates the percentage of the target sequence matching the query sequence.</p>
         </dd>
<dt>
<a id="Toplevel"></a>
  <a href="glossary/toplevel.html">Toplevel</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Toplevel%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>The largest continuous sequence for an organism. The official technical definition for <a href="glossary.html#Toplevel" title="The largest continuous sequence for an organism. The official technical definition for toplevel sequences are &#039;sequence regions in the genome assembly that are not a component of another sequence region&#039;. For example, when a genome is assembled into chromosomes, toplevel sequences will be chromosomes and unplaced scaffolds. If a genome has only been assembled into scaffolds, then toplevel sequences are scaffolds and unplaced contigs." class="lexicon-term active">toplevel</a> sequences are 'sequence regions in the genome <a href="glossary.html#Assembly" title="When the genome or transcriptome of a species is sequenced, typically many short random fragments are sequenced and reassembled by a computer algorithm into longer contiguous sequences called contigs. Genomic contigs may be assembled into longer sequences called scaffolds and sometimes, if the depth of sequencing is high enough, there may be enough information to assemble most of the scaffolds into chromosomes." class="lexicon-term active">assembly</a> that are not a component of another sequence region'. For example, when a genome is assembled into chromosomes, toplevel sequences will be chromosomes and unplaced scaffolds. If a genome has only been assembled into scaffolds, then toplevel sequences are scaffolds and unplaced contigs.</p>
         </dd>
<dt>
<a id="Transcribed_processed_pseudogene"></a>
  <a href="glossary/transcribed-processed-pseudogene.html">Transcribed processed pseudogene</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Transcribed processed pseudogene%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p><a href="glossary.html#Processed_pseudogene" title="Noncoding pseudogene produced by integration of a reverse transcribed mRNA into the genome." class="lexicon-term active">Processed pseudogene</a> with evidence of expression.</p>
         </dd>
<dt>
<a id="Transcribed_unprocessed_pseudogene"></a>
  <a href="glossary/transcribed-unprocessed-pseudogene.html">Transcribed unprocessed pseudogene</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Transcribed unprocessed pseudogene%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p><a href="glossary.html#Unprocessed_pseudogene" title="A noncoding pseudogene arising from gene duplication." class="lexicon-term active">Unprocessed pseudogene</a> with evidence of expression.</p>
         </dd>
<dt>
<a id="Transcript"></a>
  <a href="glossary/transcript.html">Transcript</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Transcript%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Nucleotide sequence resulting from the transcription of the genomic DNA to mRNA. One gene can have different transcripts or splice variants resulting from the alternative splicing of different exons in genes.</p>
         </dd>
<dt>
<a id="Transcript_ID"></a>
  <a href="glossary/transcript-id.html">Transcript ID</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Transcript ID%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Ensembl <a href="glossary.html#Transcript" title="Nucleotide sequence resulting from the transcription of the genomic DNA to mRNA. One gene can have different transcripts or splice variants resulting from the alternative splicing of different exons in genes." class="lexicon-term active">transcript</a> identifiers are unique for each splice variant.</p>
         </dd>
<dt>
<a id="translation_start_site"></a>
  <a href="glossary/translation-start-site.html">translation start site</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522translation start site%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>The position within an mRNA at which synthesis of a protein begins. The <a href="glossary.html#translation_start_site" title="The position within an mRNA at which synthesis of a protein begins. The translation start site is usually an AUG codon, but occasionally, GUG or CUG codons are used to initiate protein synthesis." class="lexicon-term active">translation start site</a> is usually an AUG <a href="glossary.html#codon" title="Three base pairs in either DNA or RNA that code for an amino acid (or stop translation)." class="lexicon-term active">codon</a>, but occasionally, GUG or CUG codons are used to initiate protein synthesis.</p>
         </dd>
<dt>
<a id="tRNA"></a>
  <a href="glossary/trna.html">tRNA</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522tRNA%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Transfer RNA. These are identified using tRNAscan. Transfer RNA (<a href="glossary.html#tRNA" title="Transfer RNA. These are identified using tRNAscan. Transfer RNA (tRNA) molecules are approximately 80 nucleotides in length. Their secondary structure includes four short double-helical elements and three loops (D, anti-codon, and T loops). Further hydrogen bonds mediate the characteristic L-shaped molecular structure. Transfer RNAs have two regions of fundamental functional importance: the anti-codon, which is responsible for specific mRNA codon recognition, and the 3&#039; end, to which the tRNA&#039;s corresponding amino acid is attached (by aminoacyl-tRNA synthetases). Transfer RNAs cope with the degeneracy of the genetic code in two manners: having more than one tRNA (with a specific anti-codon) for a particular amino acid; and &#039;wobble&#039; base-pairing, i.e. permitting non-standard base-pairing at the 3rd anti-codon position. Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000253" class="lexicon-term active">tRNA</a>) molecules are approximately 80 nucleotides in length. Their secondary structure includes four short double-helical elements and three loops (D, anti-<a href="glossary.html#codon" title="Three base pairs in either DNA or RNA that code for an amino acid (or stop translation)." class="lexicon-term active">codon</a>, and T loops). Further hydrogen bonds mediate the characteristic L-shaped molecular structure. Transfer RNAs have two regions of fundamental functional importance: the anti-codon, which is responsible for specific mRNA codon recognition, and the 3' end, to which the tRNA's corresponding amino acid is attached (by aminoacyl-tRNA synthetases). Transfer RNAs cope with the degeneracy of the genetic code in two manners: having more than one tRNA (with a specific anti-codon) for a particular amino acid; and 'wobble' base-pairing, i.e. permitting non-standard base-pairing at the 3rd anti-codon position. <a href="http://www.sequenceontology.org/browser/current_svn/term/SO:0000253">Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000253</a></p>
         </dd>
<dt>
<a id="tRNA_pseudogene"></a>
  <a href="glossary/trna-pseudogene.html">tRNA pseudogene</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522tRNA pseudogene%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Transfer RNA <a href="glossary.html#Pseudogene" title="A noncoding sequence similar to an active protein. A sequence that closely resembles a known functional gene, at another locus within a genome, that is non-functional as a consequence of (usually several) mutations that prevent either its transcription or translation (or both). In general, pseudogenes result from either reverse transcription of a transcript of their &quot;normal&quot; paralog (SO:0000043) (in which case the pseudogene typically lacks introns and includes a poly(A) tail) or from recombination (SO:0000044) (in which case the pseudogene is typically a tandem duplication of its &quot;normal&quot; paralog). Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000336" class="lexicon-term active">pseudogene</a>.</p>
         </dd>
  </dl>
                  <a id="letter_u"></a>
  <h2 class="lexicon-letter">U</h2>
  <dl>
    <dt>
<a id="Unigene"></a>
  <a href="glossary/unigene.html">Unigene</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Unigene%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p><a href="glossary.html#Unigene" title="UniGene is an experimental system for automatically partitioning GenBank sequences into a non-redundant set of gene-oriented clusters. Each Unigene cluster contains sequences that represent a unique gene, as well as related information such as the tissue types in which the gene has been expressed and map location." class="lexicon-term active">UniGene</a> is an experimental system for automatically partitioning GenBank sequences into a non-redundant set of gene-oriented clusters. Each Unigene cluster contains sequences that represent a unique gene, as well as related information such as the tissue types in which the gene has been expressed and map location.</p>
         </dd>
<dt>
<a id="UniProtSwiss-Prot"></a>
  <a href="glossary/uniprotswiss-prot.html">UniProt/Swiss-Prot</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522UniProt/Swiss-Prot%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>(Universal Protein Resource) is the world's most comprehensive catalogue of information on proteins. <a href="glossary.html#UniProtSwiss-Prot" title="(Universal Protein Resource) is the world&#039;s most comprehensive catalogue of information on proteins. UniProt/Swiss-Prot is a curated protein sequence database that provides a high level of annotation, a minimal level of redundancy and high level of integration with other databases. SwissProt is maintained collaboratively by the Swiss Institute for Bioinformatics (SIB) and the European Bioinformatics Institute (EBI)." class="lexicon-term active">UniProt/Swiss-Prot</a> is a curated protein sequence database that provides a high level of annotation, a minimal level of redundancy and high level of integration with other databases. SwissProt is maintained collaboratively by the Swiss Institute for Bioinformatics (SIB) and the European Bioinformatics Institute (EBI).</p>
         </dd>
<dt>
<a id="UniProtTrEMBL"></a>
  <a href="glossary/uniprottrembl.html">UniProt/TrEMBL</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522UniProt/TrEMBL%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>SPTrEMBL is a subset of TrEMBL (Translated EMBL database) containing the computer-annotated protein translations of all coding sequences (CDS) present in the EMBL EMBL nucleotides that are not yet incorporated into the UniProt/SwissProt database.</p>
         </dd>
<dt>
<a id="UniSTS"></a>
  <a href="glossary/unists.html">UniSTS</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522UniSTS%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p><a href="glossary.html#UniSTS" title="UniSTS is a NCBI resource for non-redundant Sequence Tagged Sites (STS) markers. For each marker, UniSTS displays the primer sequences, product size, and mapping information, as well as cross references to dbSNP, RHdb, GDB, MGD, etc. The marker report also lists GenBank and RefSeq records that contain the primer sequences determined by ePCR." class="lexicon-term active">UniSTS</a> is a NCBI resource for non-redundant Sequence Tagged Sites (STS) markers. For each marker, UniSTS displays the primer sequences, product size, and mapping information, as well as cross references to <a href="glossary.html#dbSNP" title="    The Single Nucleotide Polymorphism database (dbSNP) is a public-domain archive for a broad collection of simple (short) genetic polymorphisms. This collection of polymorphisms is maintained by NCBI and includes single-base nucleotide substitutions (also known as single nucleotide polymorphisms or SNPs), small-scale multi-base deletions or insertions (also called deletion insertion polymorphisms, indels or DIPs), and retroposable element insertions and microsatellite repeat variations (also called short tandem repeats or STRs). See more details here." class="lexicon-term active">dbSNP</a>, RHdb, GDB, MGD, etc. The marker report also lists GenBank and <a href="glossary.html#RefSeq" title="NCBI&#039;s Reference Sequences (RefSeq) database is a curated database of Genbank&#039;s genomes, mRNAs and proteins. RefSeq attempts to provide a comprehensive, integrated, non-redundant set of sequences, including genomic DNA, tRNA, and protein products, providing a stable reference for gene identification and characterization, mutation analysis, expression studies, polymorphism discovery, and comparative analyses." class="lexicon-term active">RefSeq</a> records that contain the primer sequences determined by ePCR.</p>
         </dd>
<dt>
<a id="Unitary_pseudogene"></a>
  <a href="glossary/unitary-pseudogene.html">Unitary pseudogene</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Unitary pseudogene%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>An <a href="glossary.html#Unprocessed_pseudogene" title="A noncoding pseudogene arising from gene duplication." class="lexicon-term active">unprocessed pseudogene</a> with an active orthologue in another species.</p>
         </dd>
<dt>
<a id="Unprocessed_pseudogene"></a>
  <a href="glossary/unprocessed-pseudogene.html">Unprocessed pseudogene</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Unprocessed pseudogene%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A noncoding <a href="glossary.html#Pseudogene" title="A noncoding sequence similar to an active protein. A sequence that closely resembles a known functional gene, at another locus within a genome, that is non-functional as a consequence of (usually several) mutations that prevent either its transcription or translation (or both). In general, pseudogenes result from either reverse transcription of a transcript of their &quot;normal&quot; paralog (SO:0000043) (in which case the pseudogene typically lacks introns and includes a poly(A) tail) or from recombination (SO:0000044) (in which case the pseudogene is typically a tandem duplication of its &quot;normal&quot; paralog). Source: http://www.sequenceontology.org/browser/current_svn/term/SO:0000336" class="lexicon-term active">pseudogene</a> arising from gene duplication.</p>
         </dd>
<dt>
<a id="UTR_Untranslated_Region"></a>
  <a href="glossary/utr-untranslated-region.html">UTR (Untranslated Region)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522UTR (Untranslated Region)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>The 5' UTR is the portion of an mRNA from the 5' end to the position of the first <a href="glossary.html#codon" title="Three base pairs in either DNA or RNA that code for an amino acid (or stop translation)." class="lexicon-term active">codon</a> used in translation. The 3' UTR is the portion of an mRNA from the position of the last codon that is used in translation to the 3' end.</p>
         </dd>
  </dl>
                  <a id="letter_v"></a>
  <h2 class="lexicon-letter">V</h2>
  <dl>
    <dt>
<a id="Validation_status"></a>
  <a href="glossary/validation-status.html">Validation status</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Validation status%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>A measure of confidence that a variant is a true polymorphism. It includes 1000 Genomes, HapMap and other validation statuses from <a href="glossary.html#dbSNP" title="    The Single Nucleotide Polymorphism database (dbSNP) is a public-domain archive for a broad collection of simple (short) genetic polymorphisms. This collection of polymorphisms is maintained by NCBI and includes single-base nucleotide substitutions (also known as single nucleotide polymorphisms or SNPs), small-scale multi-base deletions or insertions (also called deletion insertion polymorphisms, indels or DIPs), and retroposable element insertions and microsatellite repeat variations (also called short tandem repeats or STRs). See more details here." class="lexicon-term active">dbSNP</a> such as frequency and cluster. See a detailed description on the dbSNP website.</p>
         </dd>
<dt>
<a id="Variation_source"></a>
  <a href="glossary/variation-source.html">Variation source</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Variation source%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>The origin of the variation data (e.g. <a href="glossary.html#dbSNP" title="    The Single Nucleotide Polymorphism database (dbSNP) is a public-domain archive for a broad collection of simple (short) genetic polymorphisms. This collection of polymorphisms is maintained by NCBI and includes single-base nucleotide substitutions (also known as single nucleotide polymorphisms or SNPs), small-scale multi-base deletions or insertions (also called deletion insertion polymorphisms, indels or DIPs), and retroposable element insertions and microsatellite repeat variations (also called short tandem repeats or STRs). See more details here." class="lexicon-term active">dbSNP</a>, COSMIC, DGVa).</p>
         </dd>
<dt>
<a id="Vector_competence"></a>
  <a href="glossary/vector-competence.html">Vector competence</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Vector competence%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>The ability to transmit a pathogen, e.g. Plasmodium, dengue.</p>
         </dd>
<dt>
<a id="VEP_Variant_Effect_Predictor"></a>
  <a href="glossary/vep-variant-effect-predictor.html">VEP (Variant Effect Predictor)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522VEP (Variant Effect Predictor)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Ensembl Tool that allows users to provide a list of variants and export a results file containing consequence types.</p>
         </dd>
  </dl>
                  <a id="letter_y"></a>
  <h2 class="lexicon-letter">Y</h2>
  <dl>
    <dt>
<a id="YAC_Yeast_Artificial_Chromosome"></a>
  <a href="glossary/yac-yeast-artificial-chromosome.html">YAC (Yeast Artificial Chromosome)</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522YAC (Yeast Artificial Chromosome)%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p>Originated from a bacterial plasmid, a YAC contains a yeast centromeric region (CEN), a yeast origin of DNA replication, a cluster of unique rectriction sites and a selectable marker and a telomere region at the en of each arm. YACs are capable of cloning extremely large segments of DNA (over 1 megabase long) into a host cell, where the DNA is propagated along with the other chromosomes of the yeast cell.</p>
         </dd>
  </dl>
                  <a id="letter_z"></a>
  <h2 class="lexicon-letter">Z</h2>
  <dl>
    <dt>
<a id="Zoophilic"></a>
  <a href="glossary/zoophilic.html">Zoophilic</a>      <span class="lexicon-extralinks">
          <a href="https://legacy.vectorbase.org/search/site/%2522Zoophilic%2522?destination=glossary">search for term</a>        </span>
  </dt>
  <dd>
         <p><a href="glossary.html#Zoophilic" title="Zoophilic mosquitoes preferentially feed on animals." class="lexicon-term active">Zoophilic</a> mosquitoes preferentially feed on animals.</p>
         </dd>
  </dl>
            </div>
</div>
      </div>
      </div><!-- /inner-inner -->
	  </div><!-- /inner-wrapper -->
  </div><!-- /block-inner -->
</div><!-- /block -->    </div><!-- /content-inner -->
  </div><!-- /content -->
				                                                          			   
			
                          </div><!-- /content-region-inner -->
                        </div><!-- /content-region -->

                      </div><!-- /content-group-inner -->
                    </div><!-- /content-group -->

                                      </div><!-- /main-content-inner -->
                </div><!-- /main-content -->

                              </div><!-- /main-group-inner -->
            </div><!-- /main-group -->
          </div><!-- /main-inner -->
        </div><!-- /main -->
      </div><!-- /main-wrapper -->

            
<!-- footer region -->
<div id="footer-wrapper" class="footer-wrapper full-width clearfix">
  <div id="footer" class="region region-footer footer  grid16-16">
    <div id="footer-inner" class="footer-inner inner">
      <div id="block-block-9" class="block block-block first last odd">
  <div class="inner clearfix gutter">
  	<div class="inner-wrapper">
      <div class="inner-inner">
                        <div class="content clearfix">
         <!--<div id="bannerMessage">Not ready for the New VectorBase? <a href="http://classic.vectorbase.org/">Link back to the old site here. (not here)</a></div>-->
<!--<div id="warningMessage" style="text-align:center">
     VectorBase is currently undergoing maintenance.<br />
     Some data and/or services may be temporarily unavailable.
</div>-->
<div class="single-footer">
    <div style="text-align:center;background-color:white; border-radius: 10px;-moz-border-radius: 10px;">
        <a style="margin:5px 10px; border:none; text-decoration:none;" href="http://www.nd.edu/" ><img src="sites/default/files/ftp/site_logos/notredame.png" alt="ND" style="border:none; margin:5px 0;"/></a>
        <a style="margin:5px 10px; border:none; text-decoration:none;" href="http://www.ebi.ac.uk/" ><img src="sites/default/files/ftp/site_logos/ebi.png" alt="EBI" style="border:none; margin:5px 0;"/></a>
        <a style="margin:5px 10px; border:none; text-decoration:none;" href="http://www.imperial.ac.uk/" ><img src="sites/default/files/ftp/site_logos/imperialcollegelondon.png" alt="Imperial College" style="border:none; margin:5px 0;"/></a>
    </div>
    <p style="text-align:center;">This project has been funded in whole or in part with Federal funds from the National Institute of Allergy and Infectious Diseases, National Institutes of Health, Department of Health and Human Services, under Contract No. HHSN272201400029C.</p>
    <p>
        Bioinformatics Resource Centers for Infectious Disease: 
        <a href="https://www.vectorbase.org/" class="brc-postscript-link"><span class="brc-postscript-linktext">VectorBase</span></a> | 
        <a href="https://eupathdb.org/" class="brc-postscript-link"><span class="brc-postscript-linktext">EuPathDB</span></a> | 
        <a href="https://www.fludb.org/" class="brc-postscript-link"><span class="brc-postscript-linktext">IRD</span></a> |
        <a href="https://patricbrc.org/" class="brc-postscript-link"><span class="brc-postscript-linktext">PATRIC</span></a> |
        <a href="https://www.viprbrc.org/" class="brc-postscript-link"><span class="brc-postscript-linktext">ViPR</span></a> 
    </p>
    <div style="text-align:center">
        <a style="margin:5px 10px; border:none; text-decoration:none;" href="mailto:info@vectorbase.org" title="Email VectorBase"><img src="sites/default/files/ftp/email.png" alt="Email VectorBase" style="border:none;  height:30px; width:30px;"/></a>
        <a style="margin:5px 10px; border:none; text-decoration:none;" href="https://www.vectorbase.org/justnews/feed" title="RSS Feed"><img src="images/small_logos/48x48/feed.png" alt="RSS Feed" style="border:none;  height:30px; width:30px;"/></a>
<!--        <a style="margin:5px 10px; border:none; text-decoration:none;" href="https://www.vectorbase.org/forum" title="Forum"><img src="/sites/default/files/ftp/images/forum.png" alt="Forum" style="border:none;  height:30px; width:30px;"/></a> -->
        <a style="margin:5px 10px; border:none; text-decoration:none;" href="http://www.facebook.com/pages/VectorBase/189608142260" title="Facebook"><img src="images/small_logos/48x48/facebook.png" alt="Facebook" style="border:none; height:30px; width:30px;"/></a>
        <a style="margin:5px 10px; border:none; text-decoration:none;" href="http://twitter.com/VectorBase" title="Twitter"><img src="images/small_logos/48x48/twitter.png" alt="Twitter" style="border:none; height:30px; width:30px;"/></a>
    </div>
</div>       </div>
      </div><!-- /inner-inner -->
	  </div><!-- /inner-wrapper -->
  </div><!-- /block-inner -->
</div><!-- /block -->    </div><!-- /footer-inner -->
  </div><!-- /footer -->
</div><!-- /footer-wrapper -->
    </div><!-- /page-inner -->
  </div><!-- /page -->
  </body>

<!-- Mirrored from legacy.vectorbase.org/glossary by HTTrack Website Copier/3.x [XR&CO'2014], Mon, 10 May 2021 21:57:59 GMT -->
</html>
